<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Tempus Data Analysis 5/23/2019</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tempus 2019 Data Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Tempus.html">Tempus Data Analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a>
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Tempus Data Analysis 5/23/2019</h1>

</div>


<p><strong>Last updated:</strong> 2019-05-24</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20190207)</code> </summary></p>
<p>The command <code>set.seed(20190207)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 1da7664 </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    code/
    Ignored:    data/
    Ignored:    docs/.DS_Store
    Ignored:    docs/figure/.DS_Store
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  oncoprint.tsv

Unstaged changes:
    Modified:   analysis/Tempus.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
1da7664
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
HTML file only
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
updated analysis pre abstract
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
733bba4
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
repeat cleaned up analysis
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
b3040b6
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis neo
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
3a23302
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis 3
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
3a23302
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis 3
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 2
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 2
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 1
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
0333ca2
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
website build 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
9620b54
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="data-structure-and-format" class="section level1">
<h1>Data structure and format</h1>
<pre class="r"><code>tempusmutationtemp$GeneInfo &lt;- paste (tempusmutationtemp$variantcount, tempusmutationtemp$variant_name_C, tempusmutationtemp$variant_name_P, tempusmutationtemp$NM_number, tempusmutationtemp$typeofvariant, tempusmutationtemp$VAF, tempusmutationtemp$COSMICCases, sep=&#39;\t&#39;) # combines variant-specific variables into a temporary column for now

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo) %&gt;%mutate(maxvariantcount = max(variantcount))

tempusmutationtemp &lt;- dplyr::select(tempusmutationtemp, -c(biologicalmeaning, variant_name_C, variant_name_P, NM_number, typeofvariant, VAF, COSMICCases, variantcount, Notes)) #drops excess columns for merge purposes

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% spread(Gene,GeneInfo) #Creates individual columns for each gene with the Gene Information as the value, every variant has its own row

tempusmutationtemp$V1=NULL #Clears artificial columns that were created in the process of formatting the dataset
tempusmutationtemp$X=NULL

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo, PanelName, Normalspecimen, Normalcollecteddate, Normalreceiveddate, Tumorspecimen, Tumorcollecteddate, Tumorreceiveddate, TumorPercentage, TumorMutationalBurden, PercentileTumorMutationalBurd, MicrosatelliteInstabilityStatu, DateSigned, genecount, maxvariantcount) %&gt;% summarise_all(funs(toString(na.omit(.), sep=&#39;\t&#39;))) #This flattens all of the rows -&gt; RIGHT NOW each patient is on one line and multiple mutations for the same gene are in one entry as are their variant information for ease of merge

tempusmerge &lt;- merge (tempusclinical, tempusmutationtemp, by=&quot;StudyID&quot;, all.x=TRUE, all.y=FALSE, sort=FALSE) #Merging the clinical and mutational data into one database, reports are excluded if they do not have clinical information

tempusimmune$StudyID &lt;- tempusimmune$study_id
tempusimmunetemp &lt;- dplyr::select(tempusimmune, -c(order_id, TMB, msi_status, study_id)) #drops excess columns for merge purposes
tempusimmunetemp &lt;- filter (tempusimmunetemp, !(is.na(b))) #Removes rows with no data (as some of these are duplicates and confuse the merge process)
tempusimmunetemp &lt;- filter (tempusimmunetemp, (StudyID != &quot;PN0198&quot; &amp; StudyID != &quot;PN0206&quot;)) #Removes 2 duplicated rows in which the reason for duplication is not completely clear; this will need to be clarified later.

tempusmerge &lt;- merge (tempusmerge, tempusimmunetemp, by=&quot;StudyID&quot;, all.x=TRUE) #Merging the immune information if available with the existing dataset. Data was excluded if clinical and mutational information were not also available. </code></pre>
</div>
<div id="data-cleaningnew-variable-generation" class="section level1">
<h1>Data cleaning/new variable generation</h1>
<pre class="r"><code>tempusmerge$age50 &lt;- ifelse(tempusmerge$age_at_diagnosis &lt; 50, 1, 0) #binary variable for age &lt; 50 (which equals 1)
tempusmerge$neoadj &lt;- ifelse (((as.Date(as.character(tempusmerge$date_surg1), format=&quot;%m/%d/%Y&quot;)-as.Date(as.character(tempusmerge$date_cx1), format=&quot;%m/%d/%Y&quot;))&gt;0) &amp; (tempusmerge$Stage!=&quot;4&quot;), 1, 0) #determining if pts had neoadjuvant chemotherapy
tempusmerge$HRstatus &lt;- ifelse (tempusmerge$ER==&quot;Pos&quot; | tempusmerge$PR==&quot;Pos&quot;, 1, 0) #Hormone-receptor positive status

tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Neg&quot; &amp; tempusmerge$HRstatus==0, &quot;TNBC&quot;, NA) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Pos&quot; &amp; tempusmerge$HRstatus==0, &quot;HER+/HR-&quot;, tempusmerge$subtype) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Pos&quot; &amp; tempusmerge$HRstatus==1, &quot;HER+/HR+&quot;, tempusmerge$subtype) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Neg&quot; &amp; tempusmerge$HRstatus==1, &quot;HER-/HR+&quot;, tempusmerge$subtype) #Characterizing the subtype variable

tempusmerge$bw &lt;- ifelse (tempusmerge$race==&quot;Black&quot;, &quot;Black&quot;, NA) #Characterizing black/white alone
tempusmerge$bw &lt;- ifelse (tempusmerge$race==&quot;White&quot;, &quot;White&quot;, tempusmerge$bw) 
tempusmerge$bw &lt;- ifelse (tempusmerge$hispanic==&quot;Yes&quot;, NA, tempusmerge$bw) 

tempusmerge$vitalstatus &lt;- ifelse (tempusmerge$vitalstatus==&quot;0 Dead&quot;, &quot;0&quot;, &quot;1&quot;) #Factorizing vital status

tempusmerge$vitalstatussurv &lt;- tempusmerge$vitalstatus 

tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;1&quot;, 2, tempusmerge$vitalstatussurv)

tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;0&quot;, 1, tempusmerge$vitalstatussurv)

tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;2&quot;, 0, tempusmerge$vitalstatussurv) #switching 0 and 1 so they accurately reflect survival failure

tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1A&quot;,1,NA)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1B&quot;,1, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1&quot;,1, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2A&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2B&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3A&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3B&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3C&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;4&quot;,4, tempusmerge$stage14) #Numerical stage condensing

tempusmerge$nodeposneg &lt;- ifelse (tempusmerge$RegionalNodePositive&gt;0, 1, 0) #At least one positive lymph node

tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;00 Patient became disease-free after treatment and has not had a recurrence.&quot;, 0, 1) #Positive/negative recurrence status alone without differentiation of site. 
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;99 It is unknown whether the disease has recurred or if the patient was ever disease-free&quot;, NA, tempusmerge$recurrenceposneg) #Cleaning unknown recurrence variables from this
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;70 Since diagnosis, patient has never been disease-free. This includes cases with distant metastasis at diagnosis, systemic disease, unknown primary, or minimal disease that is not treated&quot;, NA, tempusmerge$recurrenceposneg) #Cleaning unknown recurrence variables from this

tempusmerge$TMB &lt;- tempusmerge$TumorMutationalBurden
tempusmerge$TMB &lt;- str_replace(tempusmerge$TMB, pattern = &quot; m/MB&quot;, replacement = &quot;&quot;)
tempusmerge$TMB &lt;- as.numeric(tempusmerge$TMB) #Numeric interpretation of mutational burden

tempusmerge$timefu &lt;- ifelse (tempusmerge$neoadj==1, ((as.Date(as.character(tempusmerge$date_FU), format=&quot;%m/%d/%Y&quot;))-(as.Date(as.character(tempusmerge$date_cx1), format=&quot;%m/%d/%Y&quot;)))/360, ((as.Date(as.character(tempusmerge$date_FU), format=&quot;%m/%d/%Y&quot;))-(as.Date(as.character(tempusmerge$date_surg1), format=&quot;%m/%d/%Y&quot;)))/360)
tempusmerge$timefu &lt;- ifelse(is.na(tempusmerge$timefu), ((as.Date(as.character(tempusmerge$date_FU), format=&quot;%m/%d/%Y&quot;)-as.Date(as.character(tempusmerge$date_surg1), format=&quot;%m/%d/%Y&quot;)))/360, tempusmerge$timefu)
tempusmerge$timefu &lt;- ifelse(tempusmerge$timefu&lt;0, NA, tempusmerge$timefu) #duration of follow up for survival curves in years

tempusmergedupclean &lt;- tempusmerge
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TumorPercentage, StudyID, FUN=max)==TumorPercentage),] #keep duplicates by higher %
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TMB, StudyID, FUN=max)==TMB),] #keep duplicates by higher TMB if same source, %</code></pre>
</div>
<div id="data-summary" class="section level1">
<h1>Data summary</h1>
<p>We have clinical information for 127 patients and mutation information for 131 patients. We have immune information for 70 patients.</p>
<p>We are missing clinical information for the following 4 patients. These are not included in the merged data set.</p>
<pre><code>[1] &quot;PN0118&quot; &quot;PN0124&quot; &quot;PN0153&quot; &quot;PN0155&quot;</code></pre>
<pre class="r"><code>duplicates &lt;- tempusreports %&gt;% filter(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE)) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
duplicates</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0139","2":"TL-18-1FFA96","3":"Breast","4":"0.2"},{"1":"PN0139","2":"TL-18-4BC15A","3":"Lymph node","4":"0.5"},{"1":"PN0198","2":"TL-18-200CF9","3":"Breast","4":"0.6"},{"1":"PN0198","2":"TL-18-C9E469","3":"Lymph node","4":"0.5"},{"1":"PN0264","2":"TL-18-25A755","3":"","4":"0.3"},{"1":"PN0264","2":"TL-18-18EB33","3":"","4":"0.2"},{"1":"PN0145","2":"TL-18-12B92C","3":"Breast","4":"0.4"},{"1":"PN0145","2":"TL-18-CDF9A7","3":"Lymph node","4":"0.4"},{"1":"PN0129","2":"TL-18-2F1FB7","3":"Breast","4":"0.9"},{"1":"PN0129","2":"TL-18-9BFB83","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CE9403","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CD4FA2","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-18-213530","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-19-85F69C","3":"Lymph node","4":"0.4"},{"1":"PN0263","2":"TL-18-56509D","3":"Breast","4":"0.6"},{"1":"PN0263","2":"TL-18-8A2A03","3":"Breast","4":"0.6"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>8 patients have duplicate reports (most often primary tumor and lymph node) submitted with the above characteristics. These patients are included in the merged data set but in these cases, for any analysis, we counted each patient individually/once using the specimen that offered the best tumor percentage as relevant.</p>
<p>Of note, among the immune reports, IDs PN0198 (likely due to breast and lymph node samples) and PN0206 (unclear reasons) had 2 separate reports with differing values. As such, these immune reports were excluded from the analysis.</p>
<pre class="r"><code>lymphnodes &lt;- tempusreports %&gt;% filter((Tumorspecimen==&quot;Lymph node&quot;) &amp; !(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE))) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
lymphnodes</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0162","2":"TL-18-BDD8C3","3":"Lymph node","4":"0.20"},{"1":"PN0217","2":"TL-18-DD82EF","3":"Lymph node","4":"0.20"},{"1":"PN0255","2":"TL-18-5E23D5","3":"Lymph node","4":"0.85"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>3 patients have only lymph node submitted. These patients are included in the merged data set and treated as though they had primary tumor (as this was their only tumor specimen).</p>
</div>
<div id="clinical-summary" class="section level1">
<h1>Clinical summary</h1>
<p>Low-yield variables: Date and age of enrollment (38 patients), Sex (1 male patient), Insurance (1 patient with unknown insurance, 126 patients insured), Alcohol history (90 patients not reported), gravidity/parity (26 patients reported), menopause (26 patients reported), distant metastatic sites (only 2 sites reported), histology (14 patients with at least partial lobular disease), surgical margin (4 patients with ? residual disease), 2nd primary (only one patient reported), ki67: not known in 116/127 patients</p>
<p>Variables not incorporated for this analysis: Marital status at diagnosis, breast primary site, laterality, TNM edition and clinical staging (Rather than pathologic staging), lymphovascularinvasion (64 patients only reported), radiation variables</p>
<p><img src="figure/Tempus.Rmd/Clinical%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Clinical summary-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>This is the age distribution of patients at time of diagnosis. This population skews young but does have representation throughout the age range.</p>
<table>
<caption>Distribution of patients younger than age 50 (with 0 &gt; 50 and 1 &lt; 50</caption>
<thead>
<tr class="header">
<th align="left">age&lt;50</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">70</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">57</td>
</tr>
</tbody>
</table>
<table>
<caption>Self-reported races in our cohort (Y/N refers to Hispanic ethnicity)</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Not-Hispanic</th>
<th align="right">Yes-Hispanic</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Asian</td>
<td align="right">1</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td>Black</td>
<td align="right">60</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td>White</td>
<td align="right">63</td>
<td align="right">3</td>
</tr>
</tbody>
</table>
<table>
<caption>Tobacco exposure</caption>
<thead>
<tr class="header">
<th align="left">Reported smoking history</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">00 Never used</td>
<td align="right">62</td>
</tr>
<tr class="odd">
<td align="left">01 Cigarette smoker</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">05 Previous tobacco use, but not at present time</td>
<td align="right">26</td>
</tr>
<tr class="odd">
<td align="left">99 Unknown</td>
<td align="right">20</td>
</tr>
</tbody>
</table>
<table>
<caption>Alive or dead status last known</caption>
<thead>
<tr class="header">
<th align="left">Vital status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0 Dead</td>
<td align="right">46</td>
</tr>
<tr class="even">
<td align="left">1 Alive</td>
<td align="right">81</td>
</tr>
</tbody>
</table>
<table>
<caption>Patients who experienced recurrence</caption>
<thead>
<tr class="header">
<th align="left">Recurrence</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">81</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">38</td>
</tr>
</tbody>
</table>
<table>
<caption>Fraction of patients with at least one node positive on study</caption>
<thead>
<tr class="header">
<th align="left">1+ positive nodes</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">51</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">76</td>
</tr>
</tbody>
</table>
<table>
<caption>Grade information (known for 125/127 pts)</caption>
<thead>
<tr class="header">
<th align="left">Grade</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">2</td>
<td align="right">46</td>
</tr>
<tr class="odd">
<td align="left">3</td>
<td align="right">64</td>
</tr>
<tr class="even">
<td align="left">9</td>
<td align="right">8</td>
</tr>
</tbody>
</table>
<table>
<caption>Distribution of analytical stage based on a synthesis of clinical stage and pathological stage information</caption>
<thead>
<tr class="header">
<th align="left">Compiled stage</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="right">16</td>
</tr>
<tr class="even">
<td align="left">1A</td>
<td align="right">15</td>
</tr>
<tr class="odd">
<td align="left">1B</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">2A</td>
<td align="right">37</td>
</tr>
<tr class="odd">
<td align="left">2B</td>
<td align="right">28</td>
</tr>
<tr class="even">
<td align="left">3</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">3A</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">3B</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">3C</td>
<td align="right">5</td>
</tr>
<tr class="even">
<td align="left">4</td>
<td align="right">2</td>
</tr>
</tbody>
</table>
<table>
<caption>Hormone receptor status vs HER2 status. 0 = negative and 1=positive</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Unk</th>
<th align="right">HER-neg</th>
<th align="right">HER2-pos</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0</td>
<td align="right">1</td>
<td align="right">31</td>
<td align="right">12</td>
</tr>
<tr class="even">
<td>1</td>
<td align="right">4</td>
<td align="right">53</td>
<td align="right">26</td>
</tr>
</tbody>
</table>
<table>
<caption>Overall subtype distribution</caption>
<thead>
<tr class="header">
<th align="left">Subtype</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">HER-/HR+</td>
<td align="right">53</td>
</tr>
<tr class="even">
<td align="left">HER+/HR-</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">HER+/HR+</td>
<td align="right">26</td>
</tr>
<tr class="even">
<td align="left">TNBC</td>
<td align="right">31</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of chemotherapy</caption>
<thead>
<tr class="header">
<th align="left">Chemotherapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">38</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">89</td>
</tr>
</tbody>
</table>
<table>
<caption>Patients with neoadjuvant therapy</caption>
<thead>
<tr class="header">
<th align="left">Neoadjuvant therapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">63</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">26</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of radiotherapy</caption>
<thead>
<tr class="header">
<th align="left">Radiotherapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">49</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">78</td>
</tr>
</tbody>
</table>
<table>
<caption>Initial surgery performed at time of diagnosis and initial treatment</caption>
<thead>
<tr class="header">
<th align="left">Initial surgery type</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">BCS</td>
<td align="right">48</td>
</tr>
<tr class="odd">
<td align="left">bilateral mam</td>
<td align="right">8</td>
</tr>
<tr class="even">
<td align="left">mastectomy</td>
<td align="right">68</td>
</tr>
<tr class="odd">
<td align="left">other/uk/nos</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of hormone therapy</caption>
<thead>
<tr class="header">
<th align="left">Hormone therapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">56</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">71</td>
</tr>
<tr class="odd">
<td align="left">Of note, with muta</td>
<td align="right">tion testing prior to sequencing, we had 1 patient each report a BRCA 1 and BRCA 2 mutation.</td>
</tr>
</tbody>
</table>
</div>
<div id="mutation-summary" class="section level1">
<h1>Mutation summary</h1>
<p>Variables not incorporated for this analysis: Percentile mutational burden (incomplete)</p>
<p>We have representation of 560 genes with mutations in this data set (including both primary tumors and lymph nodes). The most common mutations (incorporating BOTH pathogenic + VUS) were as follows:</p>
<pre><code>      TP53       MCL1     PIK3CA ERBB2/HER2      ZFHX3        ATM 
        72         44         39         33         27         23 
    BCORL1       FAT1      KMT2B      KMT2D 
        23         23         23         23 </code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20summary%20and%20tumor%20purity-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Mutation summary and tumor purity-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>This is the tumor purity distribution of patients at time of diagnosis. 30% purity is conisdered sufficient to perform MSI testing, for example. 104 of our 127 samples met this criteria.</p>
<pre class="r"><code>tempusoncoprint &lt;- tempusmutation %&gt;% filter(AccessionNo %in% tempusmergedupclean$AccessionNo) %&gt;% select(StudyID, AccessionNo, Gene, variant_name_P, typeofvariant)

tempusoncoprint$alteration &lt;- NA
tempusoncoprint$variant &lt;- NA

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number gain&quot; | tempusoncoprint$typeofvariant==&quot;copy numbre gain&quot; | tempusoncoprint$typeofvariant==&quot;copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy numberloss&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number Loss&quot;, &quot;CNA&quot;, tempusoncoprint$variant)

tempusoncoprint$alteration &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number gain&quot; | tempusoncoprint$typeofvariant==&quot;copy numbre gain&quot;, &quot;GAIN&quot;, tempusoncoprint$alteration)

tempusoncoprint$alteration &lt;- ifelse (tempusoncoprint$typeofvariant==&quot;copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy numberloss&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number Loss&quot;, &quot;HETLOSS&quot;, tempusoncoprint$alteration)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;frameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift VAF:&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift- LOF VAF&quot; | tempusoncoprint$typeofvariant==&quot;frameshift-LOF&quot; | tempusoncoprint$typeofvariant==&quot;FrameshiftVAF:&quot; | tempusoncoprint$typeofvariant==&quot;Inframe deletion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe deletion - GOF&quot; | tempusoncoprint$typeofvariant==&quot;inframe insertion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe insertion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe insertion - GOF&quot; | tempusoncoprint$typeofvariant==&quot;rameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift LOF&quot;, &quot;INFRAME&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;Missense  variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense varian(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;missense variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF:&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 10)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 14) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 21) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 21) - GOF VAF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 5) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant (exon 8)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant (exon 8)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant â\u0080\u0093 LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF:&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant-LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 2) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 4) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense varian(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missesne variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -LOF&quot;, &quot;MISSENSE&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;region variant&quot; | tempusoncoprint$typeofvariant==&quot;region variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice&quot; | tempusoncoprint$typeofvariant==&quot;Splice region&quot; | tempusoncoprint$typeofvariant==&quot;Splice region vairant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region vairant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;splice region variant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant -LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant â\u0080\u0093 GOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant â\u0080\u0093 LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant VAF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant-(exon 3) â\u0080\u0093 GOF&quot;, &quot;PROMOTER&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;Start loss&quot; | tempusoncoprint$typeofvariant==&quot;Start Loss&quot; | tempusoncoprint$typeofvariant==&quot;StartStop gain loss- LOF&quot; | tempusoncoprint$typeofvariant==&quot;stop gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop Gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain VAF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain-&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain-LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop loss&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain(exon 4) - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stopgain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain- LOF&quot;, &quot;TRUNC&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(is.na(tempusoncoprint$variant), &quot;OTHER&quot;, tempusoncoprint$variant)

tempusoncoprint$alteration &lt;- ifelse (is.na(tempusoncoprint$alteration), as.character(tempusoncoprint$variant_name_P), tempusoncoprint$alteration)

tempusoncoprint$alteration &lt;- ifelse (tempusoncoprint$alteration==&quot;&quot;, &quot;MUT&quot;, tempusoncoprint$alteration)

tempusoncoprint &lt;- within(tempusoncoprint, rm(StudyID, variant_name_P, typeofvariant)) #Removing alteration if not using CBioportal for Oncoprinter

write.table (tempusoncoprint, file=&quot;oncoprint.tsv&quot;, sep=&quot;\t&quot;, col.names=FALSE, row.names=FALSE)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  0.000   1.300   2.900   7.562   6.050  76.300 </code></pre>
<p><img src="figure/Tempus.Rmd/Ongoing%20mutational%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Ongoing mutational summary-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>Only 88 of 139 reports contained percentile information; this was not included for analysis. Tumor mutational burden was measured in m/MB.</p>
<table>
<caption>Distribution of MSI status</caption>
<thead>
<tr class="header">
<th align="left">MSI status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Equivocal</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">High</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">MSI can not be assessed because the tumor content is below 30%.</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">Stable</td>
<td align="right">64</td>
</tr>
<tr class="odd">
<td align="left">Stable (MSS)</td>
<td align="right">43</td>
</tr>
<tr class="even">
<td align="left">Undetermined</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td align="left">The majority of patients had stable MSI status in their tumors; on</td>
<td align="right">ly 1.5% showed MSI unstable disease.</td>
</tr>
</tbody>
</table>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.00000 0.06972 0.10595 0.11215 0.15008 0.30414      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
 0.1271  0.2409  0.2714  0.2718  0.3018  0.3944      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.03134 0.09566 0.13081 0.12943 0.16956 0.26398      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
 0.2139  0.3529  0.3711  0.3835  0.4171  0.5216      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.04802 0.06850 0.09240 0.10309 0.12424 0.22419      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.00000 0.05514 0.10400 0.11939 0.16116 0.82456      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-6.png" width="672" style="display: block; margin: auto;" /> 74 patients had “immune profile” data as calculated by Tempus. This represents a derivation from the RNA-seq data that was developed from a machine learning algorithm, NOT a pathology-based assessment. The immune score is a “support vector regression model for the immune cell estimates and formal thresholds for high or low are not in place as of yet.”</p>
</div>
<div id="recurrence-and-survival-by-ethnicity-in-our-cohort" class="section level1">
<h1>Recurrence and survival by ethnicity in our cohort</h1>
<pre class="r"><code>fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$bw, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by ethnicity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Ethnicity%20recurrence%20and%20survival-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$bw, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;percentage&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by ethnicity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Ethnicity%20recurrence%20and%20survival-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="mutations-among-different-subgroups---subtype-and-race-not-significant" class="section level1">
<h1>Mutations among different subgroups - Subtype and race (Not significant)</h1>
<pre class="r"><code>subtypeHERposHRneg &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER+/HR-&quot;)
allgenesHERposHRneg &lt;- colSums(!((subtypeHERposHRneg[,105:663])==&quot;&quot;))
allgenesHERposHRneg[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERposHRneg[&quot;ERBB2&quot;], allgenesHERposHRneg[&quot;ERBB2 (HER2)&quot;], allgenesHERposHRneg[&quot;ERBB2(HER2)&quot;], allgenesHERposHRneg[&quot;ERBB2HER2)&quot;])
headHERposHRneg &lt;- head(sort(allgenesHERposHRneg, decreasing=TRUE), 10)
headHERposHRneg</code></pre>
<pre><code>        TP53   ERBB2/HER2        CDK12        ERBB2       PIK3CA 
           9            9            6            6            4 
ERBB2 (HER2)          FLG        KDM6A         MCL1         BCOR 
           3            3            3            3            2 </code></pre>
<pre class="r"><code>subtypeHERnegHRpos &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER-/HR+&quot;)
allgenesHERnegHRpos &lt;- colSums(!((subtypeHERnegHRpos[,105:663])==&quot;&quot;))
allgenesHERnegHRpos[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERnegHRpos[&quot;ERBB2&quot;], allgenesHERnegHRpos[&quot;ERBB2 (HER2)&quot;], allgenesHERnegHRpos[&quot;ERBB2(HER2)&quot;], allgenesHERnegHRpos[&quot;ERBB2HER2)&quot;])
headHERnegHRpos &lt;- head(sort(allgenesHERnegHRpos, decreasing=TRUE), 10)
headHERnegHRpos</code></pre>
<pre><code>  MCL1 PIK3CA   TP53  ZFHX3 BCORL1  KMT2B   FAT1  KMT2D    ALK  AMER1 
    22     21     19     15     14     13     12     12     11     11 </code></pre>
<pre class="r"><code>subtypeTNBC &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;TNBC&quot;)
allgenesTNBC &lt;- colSums(!((subtypeTNBC[,105:663])==&quot;&quot;))
allgenesTNBC[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesTNBC[&quot;ERBB2&quot;], allgenesTNBC[&quot;ERBB2 (HER2)&quot;], allgenesTNBC[&quot;ERBB2(HER2)&quot;], allgenesTNBC[&quot;ERBB2HER2)&quot;])
headTNBC&lt;- head(sort(allgenesTNBC, decreasing=TRUE), 10)
headTNBC</code></pre>
<pre><code>  TP53   MCL1 NOTCH1    FLG  KMT2C    MYC  AMER1   CDK4 CDKN1B DNMT3A 
    25      8      7      5      5      5      4      4      4      4 </code></pre>
<pre class="r"><code>subtypeHERposHRpos &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER+/HR+&quot;)
allgenesHERposHRpos &lt;- colSums(!((subtypeHERposHRpos[,105:663])==&quot;&quot;))
allgenesHERposHRpos[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERposHRpos[&quot;ERBB2&quot;], allgenesHERposHRpos[&quot;ERBB2 (HER2)&quot;], allgenesHERposHRpos[&quot;ERBB2(HER2)&quot;], allgenesHERposHRpos[&quot;ERBB2HER2)&quot;])
headHERposHRpos &lt;- head(sort(allgenesHERposHRpos, decreasing=TRUE), 10)
headHERposHRpos</code></pre>
<pre><code>        TP53   ERBB2/HER2       PIK3CA ERBB2 (HER2)          ATM 
          17           12           10            9            8 
        MCL1        CDK12         RARA        AMER1        HNF1B 
           8            7            7            6            6 </code></pre>
<pre class="r"><code>chisq.test(headHERnegHRpos, headHERposHRneg, headTNBC, headHERposHRpos)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  headHERnegHRpos and headHERposHRneg
X-squared = 33.75, df = 28, p-value = 0.2092</code></pre>
<pre class="r"><code>subtypewhite &lt;- tempusmergedupclean %&gt;% filter(race==&quot;White&quot;)
allgeneswhite&lt;- colSums(!((subtypewhite[,105:663])==&quot;&quot;))
allgeneswhite[&quot;ERBB2/HER2&quot;] &lt;- sum(allgeneswhite[&quot;ERBB2&quot;], allgeneswhite[&quot;ERBB2 (HER2)&quot;], allgeneswhite[&quot;ERBB2(HER2)&quot;], allgeneswhite[&quot;ERBB2HER2)&quot;])
headwhite &lt;- head(sort(allgeneswhite, decreasing=TRUE), 5)
headwhite</code></pre>
<pre><code>      TP53       MCL1     PIK3CA ERBB2/HER2     BCORL1 
        32         25         20         14         13 </code></pre>
<pre class="r"><code>subtypeblack &lt;- tempusmergedupclean %&gt;% filter(race==&quot;Black&quot;)
allgenesblack&lt;- colSums(!((subtypeblack[,105:663])==&quot;&quot;))
allgenesblack[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesblack[&quot;ERBB2&quot;], allgenesblack[&quot;ERBB2 (HER2)&quot;], allgenesblack[&quot;ERBB2(HER2)&quot;], allgenesblack[&quot;ERBB2HER2)&quot;])
headblack &lt;- head(sort(allgenesblack, decreasing=TRUE), 5)
headblack</code></pre>
<pre><code>      TP53     PIK3CA ERBB2/HER2       MCL1      ZFHX3 
        40         19         19         18         15 </code></pre>
<pre class="r"><code>chisq.test(headwhite, headblack)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  headwhite and headblack
X-squared = 15, df = 12, p-value = 0.2414</code></pre>
<p>We did not see a significant difference in our cohort between breast cancer subtype or race and the most commonly mutated genes (outside of overrepresentation of ERBB2/HER2 in the HER2+ subtype).</p>
<pre class="r"><code>tempusmergedupclean$TP53YN &lt;- ifelse (tempusmergedupclean$TP53==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$TP53YN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$TP53YN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusmergedupclean$BRCAYN &lt;- ifelse (tempusmergedupclean$BRCA1==&quot;&quot; &amp; tempusmergedupclean$BRCA2==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$BRCAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by BRCA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$BRCAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by BRCA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusmergedupclean$ARYN &lt;- ifelse (tempusmergedupclean$AR==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$ARYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by AR mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$ARYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by AR mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusmergedupclean$PIK3CAYN &lt;- ifelse (tempusmergedupclean$PIK3CA==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$PIK3CAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by PIK3CA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$PIK3CAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by PIK3CA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-8.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="immune-profile-data-and-association-with-survival-and-recurrence" class="section level1">
<h1>Immune profile data and association with survival and recurrence</h1>
<pre class="r"><code>tempusmergedupclean$TMBhighlow &lt;- ifelse(tempusmergedupclean$TMB&gt;20, 1, 0)
  
fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$TMBhighlow, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by TMB mutation status (high&gt;20 m/MB)&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$TMBhighlow, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by TMB (high&gt;20 m/mB&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusmergedupclean$immunemedian &lt;- ifelse(tempusmergedupclean$immune&gt;0.12, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$immunemedian, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by immune score greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$recurrenceposneg)) ~tempusmergedupclean$immunemedian, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by immune score greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>surv_object &lt;- Surv(time=tempusmergedupclean$timefu, event=as.numeric(tempusmergedupclean$vitalstatussurv))
     
fit.coxph &lt;- coxph(surv_object ~ tempusmergedupclean$TMBhighlow + tempusmergedupclean$immunemedian,
                   data = tempusmergedupclean)
                   
ggforest(fit.coxph, data = tempusmergedupclean, main=&quot;Comparative hazard ratios of immune measurements in patient tumors&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>Based on Chalmers et al (<a href="https://genomemedicine.biomedcentral.com/track/pdf/10.1186/s13073-017-0424-2?site=genomemedicine.biomedcentral.com" class="uri">https://genomemedicine.biomedcentral.com/track/pdf/10.1186/s13073-017-0424-2?site=genomemedicine.biomedcentral.com</a>), we defined a high TMB as &gt;20. We saw a nonsignificant trend towards increased mutational burden and immune infiltrate being associated with worse survival in these patients.</p>
<pre class="r"><code>tempusneo &lt;- tempusmergedupclean %&gt;% filter(tempusmergedupclean$neoadj==1 &amp; !(StudyID==&quot;MB-0138&quot;) &amp; !(StudyID==&quot;MB-1020&quot;) &amp; !(StudyID==&quot;MB-1049&quot;))

kable(table(tempusneo$recurrenceposneg), caption = &quot;Neoadjuvant patients who experienced recurrence&quot;, col.names=c(&quot;Recurrence&quot;, &quot;Freq&quot;))</code></pre>
<table>
<caption>Neoadjuvant patients who experienced recurrence</caption>
<thead>
<tr class="header">
<th align="left">Recurrence</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">8</td>
</tr>
</tbody>
</table>
<pre class="r"><code>kable(table(tempusneo$vitalstatus), caption = &quot;Neoadjuvant patients who did not survive (0=died)&quot;, col.names=c(&quot;Vital status&quot;, &quot;Freq&quot;))</code></pre>
<table>
<caption>Neoadjuvant patients who did not survive (0=died)</caption>
<thead>
<tr class="header">
<th align="left">Vital status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">8</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">15</td>
</tr>
</tbody>
</table>
<pre class="r"><code>fitsurvival &lt;- survfit(Surv(time=tempusneo$timefu, as.numeric(tempusneo$vitalstatussurv)) ~tempusneo$TP53YN, 
               data=tempusneo)
ggsurvplot(fitsurvival, data=tempusneo, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of neoadjuvant patients by TP53 status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Neoadjuvant%20patients-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Neoadjuvant patients-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusneo$timefu, as.numeric(tempusneo$recurrenceposneg)) ~tempusneo$TP53YN, 
               data=tempusneo)
ggsurvplot(fitrecurrence, data=tempusneo, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in neoadjuvant patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Neoadjuvant%20patients-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Neoadjuvant patients-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="heterogeneity-between-sentinel-lymph-nodes-and-primary-tumors" class="section level1">
<h1>Heterogeneity between sentinel lymph nodes and primary tumors</h1>
<pre class="r"><code>tempusbreastlymph &lt;- tempusmerge %&gt;% filter((StudyID %in% duplicates$StudyID) &amp; !(Tumorspecimen==&quot;&quot;) &amp; !(StudyID==&quot;PN0263&quot;))
  subtypelymph &lt;- tempusbreastlymph %&gt;% filter(Tumorspecimen==&quot;Lymph node&quot;)
  subtypebreast &lt;- tempusbreastlymph %&gt;% filter(Tumorspecimen==&quot;Breast&quot;)

  allgenesbreast&lt;- colSums(!((subtypebreast[,105:663])==&quot;&quot;))
  allgenesbreast[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesbreast[&quot;ERBB2&quot;], allgenesbreast[&quot;ERBB2 (HER2)&quot;], allgenesbreast[&quot;ERBB2(HER2)&quot;], allgenesbreast[&quot;ERBB2HER2)&quot;])
  print(head(sort(allgenesbreast, decreasing=TRUE)))</code></pre>
<pre><code> TP53  FAT1  MCL1 ZFHX3  ELF3 GATA1 
    5     4     4     4     3     3 </code></pre>
<pre class="r"><code>  allgeneslymph&lt;- colSums(!((subtypelymph[,105:663])==&quot;&quot;))
  allgeneslymph[&quot;ERBB2/HER2&quot;] &lt;- sum(allgeneslymph[&quot;ERBB2&quot;], allgeneslymph[&quot;ERBB2 (HER2)&quot;], allgeneslymph[&quot;ERBB2(HER2)&quot;], allgeneslymph[&quot;ERBB2HER2)&quot;])
  print(head(sort(allgeneslymph, decreasing=TRUE)))</code></pre>
<pre><code>PIK3CA   TP53 ARID1A ARID5B  ASXL1    ATM 
     4      2      1      1      1      1 </code></pre>
<p>We did see heterogeneity between the mutations represented in the lymph nodes and the mutations represented in the primary breast tumors for those samples. Two of the lymph nodes had 20% purity, 2 had 40% purity and 2 had 50% purity.</p>
</div>
<div id="extremely-mutated-patients-x-2---case-examples" class="section level1">
<h1>Extremely mutated patients x 2 - case examples</h1>
<pre class="r"><code>tempusmergedupclean$StudyID[tempusmergedupclean$genecount==max(tempusmergedupclean$genecount)]</code></pre>
<pre><code>[1] PN0268
127 Levels: MB-0009 MB-0038 MB-0084 MB-0118 MB-0122 MB-0132 ... PN0308</code></pre>
<p>The patient who had the most unique genes mutated in her tumor in our sample (with 184) was black, was living as of last f/u with iniital diagnosis in 2001 and f/u in 2011. HR+/HER2- disease without known recurrence. Her TMB was 71.3 and in the 100th percentile, MSI status was equivocal. The patient who had the highest number of variants (9) within a single gene was white, was dead with initial diagnosis in 2000 and f/u in 2006 with HR+, HER2 unknown disease with unknown recurrence status. Her TMB was 58.3 and in the 100th percentile; genecount was 3rd highest in the group, MSI status was normal. Both patients were cigarette smokers although cigarette smoking was not significantly associated with any of our mutation variables.</p>
<pre class="r"><code>sameptsamegenemultivariant &lt;- tempusmutation %&gt;% group_by (StudyID, Gene) %&gt;% filter((duplicated(StudyID) &amp; duplicated(Gene) &amp; duplicated(AccessionNo)) | (duplicated(StudyID, fromLast=TRUE) &amp; duplicated(AccessionNo, fromLast=TRUE) &amp; duplicated(Gene, fromLast=TRUE))) %&gt;% dplyr::select(&quot;StudyID&quot;, &quot;AccessionNo&quot;, &quot;Tumorspecimen&quot;, &quot;Gene&quot;, &quot;variant_name_C&quot;, &quot;variant_name_P&quot;, &quot;NM_number&quot;, &quot;typeofvariant&quot;, &quot;genecount&quot;, &quot;variantcount&quot;)</code></pre>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] bindrcpp_0.2.2  stringr_1.3.1   survminer_0.4.3 ggpubr_0.2     
 [5] magrittr_1.5    survival_2.43-3 ggplot2_3.1.0   knitr_1.21     
 [9] dplyr_0.7.8     tidyr_0.8.2    

loaded via a namespace (and not attached):
 [1] zoo_1.8-4         tidyselect_0.2.5  xfun_0.4         
 [4] purrr_0.2.5       splines_3.5.0     lattice_0.20-38  
 [7] colorspace_1.4-0  generics_0.0.2    htmltools_0.3.6  
[10] yaml_2.2.0        survMisc_0.5.5    rlang_0.3.1      
[13] R.oo_1.22.0       pillar_1.3.1      glue_1.3.0       
[16] withr_2.1.2       R.utils_2.7.0     bindr_0.1.1      
[19] plyr_1.8.4        munsell_0.5.0     gtable_0.2.0     
[22] workflowr_1.1.1   R.methodsS3_1.7.1 evaluate_0.12    
[25] labeling_0.3      highr_0.7         broom_0.5.1      
[28] Rcpp_1.0.0        xtable_1.8-3      scales_1.0.0     
[31] backports_1.1.3   jsonlite_1.6      cmprsk_2.2-7     
[34] km.ci_0.5-2       gridExtra_2.3     digest_0.6.18    
[37] stringi_1.2.4     KMsurv_0.1-5      cowplot_0.9.4    
[40] grid_3.5.0        rprojroot_1.3-2   tools_3.5.0      
[43] lazyeval_0.2.1    tibble_2.0.1      crayon_1.3.4     
[46] whisker_0.3-2     pkgconfig_2.0.2   Matrix_1.2-15    
[49] data.table_1.12.0 assertthat_0.2.0  rmarkdown_1.11   
[52] R6_2.3.0          nlme_3.1-137      git2r_0.24.0     
[55] compiler_3.5.0   </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
