<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Tempus Data Analysis 5/29/2019</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tempus 2019 Data Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Tempus.html">Tempus Data Analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a>
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Tempus Data Analysis 5/29/2019</h1>

</div>


<p><strong>Last updated:</strong> 2019-05-29</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20190207)</code> </summary></p>
<p>The command <code>set.seed(20190207)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 885580a </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    code/
    Ignored:    data/
    Ignored:    docs/.DS_Store
    Ignored:    docs/figure/.DS_Store
    Ignored:    output/.DS_Store

Unstaged changes:
    Modified:   analysis/Tempus.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
<td style="text-align:left;">
5/28
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
<td style="text-align:left;">
5/28
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
<td style="text-align:left;">
fixed survival curves
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
<td style="text-align:left;">
fixed survival curves
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
1da7664
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
HTML file only
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
updated analysis pre abstract
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
733bba4
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
<td style="text-align:left;">
repeat cleaned up analysis
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
b3040b6
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis neo
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
3a23302
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis 3
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
3a23302
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis 3
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 2
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 2
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
<td style="text-align:left;">
analysis step 1
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
0333ca2
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
website build 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
9620b54
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="data-structure-and-format" class="section level1">
<h1>Data structure and format</h1>
<pre class="r"><code>tempusmutationtemp$GeneInfo &lt;- paste (tempusmutationtemp$variantcount, tempusmutationtemp$variant_name_C, tempusmutationtemp$variant_name_P, tempusmutationtemp$NM_number, tempusmutationtemp$typeofvariant, tempusmutationtemp$VAF, tempusmutationtemp$COSMICCases, sep=&#39;\t&#39;) # combines variant-specific variables into a temporary column for now

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo) %&gt;%mutate(maxvariantcount = max(variantcount)) #Highest number of variants for an individual gene

tempusmutationtemp &lt;- dplyr::select(tempusmutationtemp, -c(biologicalmeaning, variant_name_C, variant_name_P, NM_number, typeofvariant, VAF, COSMICCases, variantcount, Notes)) #drops excess columns for merge purposes

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% spread(Gene,GeneInfo) #Creates individual columns for each gene with the Gene Information as the value, every variant has its own row

tempusmutationtemp$V1=NULL #Clears artificial columns that were created in the process of formatting the dataset
tempusmutationtemp$X=NULL

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo, PanelName, Normalspecimen, Normalcollecteddate, Normalreceiveddate, Tumorspecimen, Tumorcollecteddate, Tumorreceiveddate, TumorPercentage, TumorMutationalBurden, PercentileTumorMutationalBurd, MicrosatelliteInstabilityStatu, DateSigned, genecount, maxvariantcount) %&gt;% summarise_all(funs(toString(na.omit(.), sep=&#39;\t&#39;))) #This flattens all of the rows -&gt; RIGHT NOW each patient is on one line and multiple mutations for the same gene are in one entry as are their variant information for ease of merge

tempusmerge &lt;- merge (tempusclinical, tempusmutationtemp, by=&quot;StudyID&quot;, all.x=TRUE, all.y=FALSE, sort=FALSE) #Merging the clinical and mutational data into one database, reports are excluded if they do not have clinical information

tempusimmune$StudyID &lt;- tempusimmune$study_id
tempusimmunetemp &lt;- dplyr::select(tempusimmune, -c(order_id, TMB, msi_status, study_id)) #drops excess columns for merge purposes
tempusimmunetemp &lt;- filter (tempusimmunetemp, !(is.na(b))) #Removes rows with no data (as some of these are duplicates and confuse the merge process)
tempusimmunetemp &lt;- filter (tempusimmunetemp, (StudyID != &quot;PN0198&quot; &amp; StudyID != &quot;PN0206&quot;)) #Removes 2 duplicated rows in which the reason for duplication is not completely clear; this will need to be clarified later.

tempusmerge &lt;- merge (tempusmerge, tempusimmunetemp, by=&quot;StudyID&quot;, all.x=TRUE) #Merging the immune information if available with the existing dataset. Data was excluded if clinical and mutational information were not also available. </code></pre>
</div>
<div id="data-cleaningnew-variable-generation" class="section level1">
<h1>Data cleaning/new variable generation</h1>
<pre class="r"><code>tempusmerge$age50 &lt;- ifelse(tempusmerge$age_at_diagnosis &lt; 50, 1, 0) #binary variable for age &lt; 50 (which equals 1)
tempusmerge$neoadj &lt;- ifelse (((as.Date(as.character(tempusmerge$date_surg1), format=&quot;%m/%d/%Y&quot;)-as.Date(as.character(tempusmerge$date_cx1), format=&quot;%m/%d/%Y&quot;))&gt;0) &amp; (tempusmerge$Stage!=&quot;4&quot;), 1, 0) #determining if pts had neoadjuvant chemotherapy
tempusmerge$HRstatus &lt;- ifelse (tempusmerge$ER==&quot;Pos&quot; | tempusmerge$PR==&quot;Pos&quot;, 1, 0) #Hormone-receptor positive status

tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Neg&quot; &amp; tempusmerge$HRstatus==0, &quot;TNBC&quot;, NA) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Pos&quot; &amp; tempusmerge$HRstatus==0, &quot;HER+/HR-&quot;, tempusmerge$subtype) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Pos&quot; &amp; tempusmerge$HRstatus==1, &quot;HER+/HR+&quot;, tempusmerge$subtype) #Characterizing the subtype variable
tempusmerge$subtype &lt;- ifelse (tempusmerge$HER2.x==&quot;Neg&quot; &amp; tempusmerge$HRstatus==1, &quot;HER-/HR+&quot;, tempusmerge$subtype) #Characterizing the subtype variable

tempusmerge$bw &lt;- ifelse (tempusmerge$race==&quot;Black&quot;, &quot;Black&quot;, NA) #Characterizing black/white alone
tempusmerge$bw &lt;- ifelse (tempusmerge$race==&quot;White&quot;, &quot;White&quot;, tempusmerge$bw) 
tempusmerge$bw &lt;- ifelse (tempusmerge$hispanic==&quot;Yes&quot;, NA, tempusmerge$bw) 

tempusmerge$vitalstatus &lt;- ifelse (tempusmerge$vitalstatus==&quot;0 Dead&quot;, &quot;0&quot;, &quot;1&quot;) #Factorizing vital status
tempusmerge$vitalstatussurv &lt;- tempusmerge$vitalstatus 
tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;1&quot;, 2, tempusmerge$vitalstatussurv)
tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;0&quot;, 1, tempusmerge$vitalstatussurv)
tempusmerge$vitalstatussurv &lt;- ifelse (tempusmerge$vitalstatussurv==&quot;2&quot;, 0, tempusmerge$vitalstatussurv) #switching 0 and 1 so they accurately reflect survival failure

tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1A&quot;,1,NA)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1B&quot;,1, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;1&quot;,1, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2A&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;2B&quot;,2, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3A&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3B&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;3C&quot;,3, tempusmerge$stage14)
tempusmerge$stage14 &lt;- ifelse (tempusmerge$Stage %in% &quot;4&quot;,4, tempusmerge$stage14) #Numerical stage condensing

tempusmerge$nodeposneg &lt;- ifelse (tempusmerge$RegionalNodePositive&gt;0, 1, 0) #At least one positive lymph node

tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;00 Patient became disease-free after treatment and has not had a recurrence.&quot;, 0, 1) #Positive/negative recurrence status alone without differentiation of site. 
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;99 It is unknown whether the disease has recurred or if the patient was ever disease-free&quot;, NA, tempusmerge$recurrenceposneg) #Cleaning unknown recurrence variables from this
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;70 Since diagnosis, patient has never been disease-free. This includes cases with distant metastasis at diagnosis, systemic disease, unknown primary, or minimal disease that is not treated&quot;, NA, tempusmerge$recurrenceposneg) #Cleaning unknown recurrence variables from this

tempusmerge$pfs &lt;- tempusmerge$vitalstatussurv
tempusmerge$pfs &lt;- ifelse (tempusmerge$recurrenceposneg==1, 1, tempusmerge$pfs)
tempusmerge$pfs &lt;- ifelse (is.na(tempusmerge$recurrenceposneg) &amp; tempusmerge$vitalstatussurv==1, 1, tempusmerge$pfs)

tempusmerge$TMB &lt;- tempusmerge$TumorMutationalBurden
tempusmerge$TMB &lt;- str_replace(tempusmerge$TMB, pattern = &quot; m/MB&quot;, replacement = &quot;&quot;)
tempusmerge$TMB &lt;- as.numeric(tempusmerge$TMB) #Numeric interpretation of mutational burden

tempusmerge$timefu &lt;- ((as.Date(as.character(tempusmerge$date_FU), format=&quot;%m/%d/%y&quot;))-(as.Date(as.character(tempusmerge$datedx), format=&quot;%m/%d/%y&quot;)))/365.25

tempusmerge$pfstimefu &lt;- tempusmerge$timefu

tempusmerge$pfstimefu &lt;- ifelse (tempusmerge$pfs==1 &amp; tempusmerge$recurrenceposneg==1, ((as.Date(as.character(tempusmerge$dateRec), format=&quot;%m/%d/%y&quot;))-(as.Date(as.character(tempusmerge$datedx), format=&quot;%m/%d/%y&quot;)))/365.25, tempusmerge$pfstimefu)
#duration of follow up for survival curves in years -&gt; timefu is overall followup time and pfstimefu is followup time for progression free survival

tempusmergedupclean &lt;- tempusmerge
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TumorPercentage, StudyID, FUN=max)==TumorPercentage),] #keep duplicates by higher %
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TMB, StudyID, FUN=max)==TMB),] #keep duplicates by higher TMB if same source, %</code></pre>
</div>
<div id="data-summary" class="section level1">
<h1>Data summary</h1>
<p>We have clinical information for 127 patients and mutation information for 131 patients. We have immune information for 70 patients.</p>
<p>We are missing clinical information for the following 4 patients. These are not included in the merged data set.</p>
<pre><code>[1] &quot;PN0118&quot; &quot;PN0124&quot; &quot;PN0153&quot; &quot;PN0155&quot;</code></pre>
<pre class="r"><code>duplicates &lt;- tempusreports %&gt;% filter(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE)) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
duplicates</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0139","2":"TL-18-1FFA96","3":"Breast","4":"0.2"},{"1":"PN0139","2":"TL-18-4BC15A","3":"Lymph node","4":"0.5"},{"1":"PN0198","2":"TL-18-200CF9","3":"Breast","4":"0.6"},{"1":"PN0198","2":"TL-18-C9E469","3":"Lymph node","4":"0.5"},{"1":"PN0264","2":"TL-18-25A755","3":"","4":"0.3"},{"1":"PN0264","2":"TL-18-18EB33","3":"","4":"0.2"},{"1":"PN0145","2":"TL-18-12B92C","3":"Breast","4":"0.4"},{"1":"PN0145","2":"TL-18-CDF9A7","3":"Lymph node","4":"0.4"},{"1":"PN0129","2":"TL-18-2F1FB7","3":"Breast","4":"0.9"},{"1":"PN0129","2":"TL-18-9BFB83","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CE9403","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CD4FA2","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-18-213530","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-19-85F69C","3":"Lymph node","4":"0.4"},{"1":"PN0263","2":"TL-18-56509D","3":"Breast","4":"0.6"},{"1":"PN0263","2":"TL-18-8A2A03","3":"Breast","4":"0.6"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>8 patients have duplicate reports (most often primary tumor and lymph node) submitted with the above characteristics. These patients are included in the merged data set but in these cases, for any analysis, we counted each patient individually/once using the specimen that offered the best tumor percentage as relevant.</p>
<p>Of note, among the immune reports, IDs PN0198 (likely due to breast and lymph node samples) and PN0206 (unclear reasons) had 2 separate reports with differing values. As such, these immune reports were excluded from the analysis.</p>
<pre class="r"><code>lymphnodes &lt;- tempusreports %&gt;% filter((Tumorspecimen==&quot;Lymph node&quot;) &amp; !(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE))) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
lymphnodes</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0162","2":"TL-18-BDD8C3","3":"Lymph node","4":"0.20"},{"1":"PN0217","2":"TL-18-DD82EF","3":"Lymph node","4":"0.20"},{"1":"PN0255","2":"TL-18-5E23D5","3":"Lymph node","4":"0.85"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>3 patients have only lymph node submitted. These patients are included in the merged data set and treated as though they had primary tumor (as this was their only tumor specimen).</p>
</div>
<div id="clinical-summary" class="section level1">
<h1>Clinical summary</h1>
<p>Low-yield variables: Date and age of enrollment (38 patients), Sex (1 male patient), Insurance (1 patient with unknown insurance, 126 patients insured), Alcohol history (90 patients not reported), gravidity/parity (26 patients reported), menopause (26 patients reported), distant metastatic sites (only 2 sites reported), histology (14 patients with at least partial lobular disease), surgical margin (4 patients with ? residual disease), 2nd primary (only one patient reported), ki67: not known in 116/127 patients</p>
<p>Variables not incorporated for this analysis: Marital status at diagnosis, breast primary site, laterality, TNM edition and clinical staging (Rather than pathologic staging), lymphovascularinvasion (64 patients only reported), radiation variables</p>
<p><img src="figure/Tempus.Rmd/Clinical%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Clinical summary-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>This is the age distribution of patients at time of diagnosis. This population skews young but does have representation throughout the age range.</p>
<table>
<caption>Distribution of patients younger than age 50 (with 0 &gt; 50 and 1 &lt; 50</caption>
<thead>
<tr class="header">
<th align="left">age&lt;50</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">70</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">57</td>
</tr>
</tbody>
</table>
<table>
<caption>Self-reported races in our cohort (Y/N refers to Hispanic ethnicity)</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Not-Hispanic</th>
<th align="right">Yes-Hispanic</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Asian</td>
<td align="right">1</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td>Black</td>
<td align="right">60</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td>White</td>
<td align="right">63</td>
<td align="right">3</td>
</tr>
</tbody>
</table>
<table>
<caption>Tobacco exposure</caption>
<thead>
<tr class="header">
<th align="left">Reported smoking history</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">00 Never used</td>
<td align="right">62</td>
</tr>
<tr class="odd">
<td align="left">01 Cigarette smoker</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">05 Previous tobacco use, but not at present time</td>
<td align="right">26</td>
</tr>
<tr class="odd">
<td align="left">99 Unknown</td>
<td align="right">20</td>
</tr>
</tbody>
</table>
<table>
<caption>Alive or dead status last known</caption>
<thead>
<tr class="header">
<th align="left">Vital status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0 Dead</td>
<td align="right">46</td>
</tr>
<tr class="even">
<td align="left">1 Alive</td>
<td align="right">81</td>
</tr>
</tbody>
</table>
<table>
<caption>Patients who experienced recurrence</caption>
<thead>
<tr class="header">
<th align="left">Recurrence</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">81</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">38</td>
</tr>
</tbody>
</table>
<table>
<caption>Fraction of patients with at least one node positive on study</caption>
<thead>
<tr class="header">
<th align="left">1+ positive nodes</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">51</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">76</td>
</tr>
</tbody>
</table>
<table>
<caption>Grade information (known for 125/127 pts)</caption>
<thead>
<tr class="header">
<th align="left">Grade</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">2</td>
<td align="right">46</td>
</tr>
<tr class="odd">
<td align="left">3</td>
<td align="right">64</td>
</tr>
<tr class="even">
<td align="left">9</td>
<td align="right">8</td>
</tr>
</tbody>
</table>
<table>
<caption>Distribution of analytical stage based on a synthesis of clinical stage and pathological stage information</caption>
<thead>
<tr class="header">
<th align="left">Compiled stage</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="right">16</td>
</tr>
<tr class="even">
<td align="left">1A</td>
<td align="right">15</td>
</tr>
<tr class="odd">
<td align="left">1B</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">2A</td>
<td align="right">37</td>
</tr>
<tr class="odd">
<td align="left">2B</td>
<td align="right">28</td>
</tr>
<tr class="even">
<td align="left">3</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">3A</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">3B</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">3C</td>
<td align="right">5</td>
</tr>
<tr class="even">
<td align="left">4</td>
<td align="right">2</td>
</tr>
</tbody>
</table>
<table>
<caption>Hormone receptor status vs HER2 status. 0 = negative and 1=positive</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Unk</th>
<th align="right">HER-neg</th>
<th align="right">HER2-pos</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0</td>
<td align="right">1</td>
<td align="right">31</td>
<td align="right">12</td>
</tr>
<tr class="even">
<td>1</td>
<td align="right">4</td>
<td align="right">53</td>
<td align="right">26</td>
</tr>
</tbody>
</table>
<table>
<caption>Overall subtype distribution</caption>
<thead>
<tr class="header">
<th align="left">Subtype</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">HER-/HR+</td>
<td align="right">53</td>
</tr>
<tr class="even">
<td align="left">HER+/HR-</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">HER+/HR+</td>
<td align="right">26</td>
</tr>
<tr class="even">
<td align="left">TNBC</td>
<td align="right">31</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of chemotherapy</caption>
<thead>
<tr class="header">
<th align="left">Chemotherapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">38</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">89</td>
</tr>
</tbody>
</table>
<table>
<caption>Patients with neoadjuvant therapy</caption>
<thead>
<tr class="header">
<th align="left">Neoadjuvant therapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">63</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">26</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of radiotherapy</caption>
<thead>
<tr class="header">
<th align="left">Radiotherapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">49</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">78</td>
</tr>
</tbody>
</table>
<table>
<caption>Initial surgery performed at time of diagnosis and initial treatment</caption>
<thead>
<tr class="header">
<th align="left">Initial surgery type</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">BCS</td>
<td align="right">48</td>
</tr>
<tr class="odd">
<td align="left">bilateral mam</td>
<td align="right">8</td>
</tr>
<tr class="even">
<td align="left">mastectomy</td>
<td align="right">68</td>
</tr>
<tr class="odd">
<td align="left">other/uk/nos</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of hormone therapy</caption>
<thead>
<tr class="header">
<th align="left">Hormone therapy</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">56</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">71</td>
</tr>
<tr class="odd">
<td align="left">Of note, with muta</td>
<td align="right">tion testing prior to sequencing, we had 1 patient each report a BRCA 1 and BRCA 2 mutation.</td>
</tr>
</tbody>
</table>
</div>
<div id="mutation-summary" class="section level1">
<h1>Mutation summary</h1>
<p>Variables not incorporated for this analysis: Percentile mutational burden (incomplete)</p>
<p>We have representation of 560 genes with mutations in this data set (including both primary tumors and lymph nodes). The most common mutations (incorporating BOTH pathogenic + VUS) were as follows:</p>
<pre><code>      TP53       MCL1     PIK3CA ERBB2/HER2      ZFHX3        ATM 
        72         44         39         33         27         23 
    BCORL1       FAT1      KMT2B      KMT2D     NOTCH1      AMER1 
        23         23         23         23         23         22 
     CDK12      KDM5C        FLG        MYC     NOTCH3       TSC2 
        21         21         20         20         20         19 
       ALK       CUX1      GATA3      HNF1B       SPEN     ARID1A 
        18         18         18         18         18         17 
    BCL11B       BCOR        CIC       CDH1 KMT2C/MLL3       ATRX 
        17         17         17         16         16         15 </code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20summary%20and%20tumor%20purity-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Mutation summary and tumor purity-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>This is the tumor purity distribution of patients at time of diagnosis. 30% purity is conisdered sufficient to perform MSI testing, for example. 104 of our 127 samples met this criteria.</p>
<pre class="r"><code>tempusoncoprint &lt;- tempusmutation 

tempusoncoprint$alteration &lt;- NA
tempusoncoprint$variant &lt;- NA

#tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number gain&quot; | tempusoncoprint$typeofvariant==&quot;copy numbre gain&quot; | tempusoncoprint$typeofvariant==&quot;copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy numberloss&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number Loss&quot;, &quot;CNA&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy number gain&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number gain&quot; | tempusoncoprint$typeofvariant==&quot;copy numbre gain&quot;, &quot;CNAGAIN&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse (tempusoncoprint$typeofvariant==&quot;copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy number loss&quot; | tempusoncoprint$typeofvariant==&quot;Copy numberloss&quot; | tempusoncoprint$typeofvariant==&quot;Copy Number Loss&quot;, &quot;CNALOSS&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;frameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift VAF:&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift- LOF VAF&quot; | tempusoncoprint$typeofvariant==&quot;frameshift-LOF&quot; | tempusoncoprint$typeofvariant==&quot;FrameshiftVAF:&quot; | tempusoncoprint$typeofvariant==&quot;Inframe deletion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe deletion - GOF&quot; | tempusoncoprint$typeofvariant==&quot;inframe insertion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe insertion&quot; | tempusoncoprint$typeofvariant==&quot;Inframe insertion - GOF&quot; | tempusoncoprint$typeofvariant==&quot;rameshift&quot; | tempusoncoprint$typeofvariant==&quot;Frameshift LOF&quot;, &quot;INFRAME&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;Missense  variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense varian(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;missense variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF:&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 10)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 14) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 21) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 21) - GOF VAF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant (exon 5) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant (exon 8)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense Variant (exon 8)- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant â\u0080\u0093 LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF:&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant-LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 2) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant(exon 4) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense varian(exon 10) - GOF&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Missesne variant&quot; | tempusoncoprint$typeofvariant==&quot;Missense variant -LOF&quot;, &quot;MISSENSE&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;region variant&quot; | tempusoncoprint$typeofvariant==&quot;region variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice&quot; | tempusoncoprint$typeofvariant==&quot;Splice region&quot; | tempusoncoprint$typeofvariant==&quot;Splice region vairant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region vairant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;splice region variant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant -LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant â\u0080\u0093 GOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant â\u0080\u0093 LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant VAF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant- LOF&quot; | tempusoncoprint$typeofvariant==&quot;Splice region variant-(exon 3) â\u0080\u0093 GOF&quot;, &quot;PROMOTER&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(tempusoncoprint$typeofvariant==&quot;Start loss&quot; | tempusoncoprint$typeofvariant==&quot;Start Loss&quot; | tempusoncoprint$typeofvariant==&quot;StartStop gain loss- LOF&quot; | tempusoncoprint$typeofvariant==&quot;stop gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop Gain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain VAF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain-&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain- GOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain-LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stop loss&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain(exon 4) - LOF&quot; | tempusoncoprint$typeofvariant==&quot;Stopgain&quot; | tempusoncoprint$typeofvariant==&quot;Stop gain- LOF&quot;, &quot;TRUNC&quot;, tempusoncoprint$variant)

tempusoncoprint$variant &lt;- ifelse(is.na(tempusoncoprint$variant), &quot;OTHER&quot;, tempusoncoprint$variant)

#tempusoncoprint$alteration &lt;- ifelse (is.na(tempusoncoprint$alteration), as.character(tempusoncoprint$variant_name_P), tempusoncoprint$alteration)

#tempusoncoprint$alteration &lt;- ifelse (tempusoncoprint$alteration==&quot;&quot;, &quot;MUT&quot;, tempusoncoprint$alteration)

#tempusoncoprint$varinfo &lt;- paste (tempusoncoprint$variant, tempusoncoprint$alteration, sep=&#39;,&#39;)

tempusoncoprint$caseID = paste (tempusoncoprint$StudyID, tempusoncoprint$AccessionNo)

subtype &lt;- tempusmergedupclean %&gt;% select(StudyID, AccessionNo, subtype)
tempusoncoprint &lt;- merge(tempusoncoprint, subtype, all.x=TRUE)

germline &lt;- tempusoncoprint %&gt;% filter (VAF &gt; 0.5) %&gt;% select (StudyID, AccessionNo, TumorPercentage, VAF, Gene, variant_name_P) #List of possible germline variants given high allele frequency
cleanednames &lt;- tempusmergedupclean %&gt;% select(StudyID, AccessionNo)
germline &lt;- merge(germline, cleanednames)
germline</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["VAF"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Gene"],"name":[5],"type":["fctr"],"align":["left"]},{"label":["variant_name_P"],"name":[6],"type":["fctr"],"align":["left"]}],"data":[{"1":"MB-0038","2":"TL-18-A9A9C2","3":"0.70","4":"0.5230","5":"ERBB2 (HER2)","6":"p.L755S"},{"1":"MB-0084","2":"TL-18-25A3C4","3":"0.80","4":"0.6730","5":"TP53","6":"p.G266E"},{"1":"MB-0118","2":"TL-18-F67579","3":"0.70","4":"0.6366","5":"TP53","6":"p.L145P"},{"1":"MB-0118","2":"TL-18-F67579","3":"0.70","4":"0.5564","5":"PTEN","6":"p.W111G"},{"1":"MB-0122","2":"TL-18-C46E0B","3":"0.80","4":"0.6950","5":"ATRX","6":""},{"1":"MB-0144","2":"TL-18-9D428E","3":"0.50","4":"0.5680","5":"TP53","6":""},{"1":"MB-0154","2":"TL-18-93AD89","3":"0.70","4":"0.6284","5":"TP53","6":"p.R248Q"},{"1":"MB-0219","2":"TL-18-DB0F79","3":"0.90","4":"0.7751","5":"TP53","6":"p.R342fs"},{"1":"MB-0219","2":"TL-18-DB0F79","3":"0.90","4":"0.7598","5":"SMC1A","6":"p.A616V"},{"1":"MB-0316","2":"TL-18-560AB2","3":"0.70","4":"0.6331","5":"RAF1","6":"p.R354W"},{"1":"MB-0316","2":"TL-18-560AB2","3":"0.70","4":"0.5795","5":"ATM","6":"p.V716L"},{"1":"MB-0316","2":"TL-18-560AB2","3":"0.70","4":"0.5810","5":"TP53","6":"p.R209fs"},{"1":"MB-0459","2":"TL-18-84DCF0","3":"0.80","4":"0.6816","5":"BRCA1","6":""},{"1":"MB-0593","2":"TL-18-52D9D6","3":"0.70","4":"0.6182","5":"LAG3","6":"p.R210Q"},{"1":"MB-0597","2":"TL-18-1D9668","3":"0.60","4":"0.6260","5":"CDH1","6":"p.N428fs"},{"1":"MB-0650","2":"TL-18-F5769F","3":"0.70","4":"0.6418","5":"FAM175A","6":""},{"1":"MB-0720","2":"TL-18-228825","3":"0.90","4":"0.6060","5":"SMAD2","6":"p.S464*"},{"1":"MB-0720","2":"TL-18-228825","3":"0.90","4":"0.8480","5":"ERBB2 (HER2)","6":"p.A775_G776insYVMA"},{"1":"MB-0739","2":"TL-18-7E6117","3":"0.70","4":"0.5274","5":"BCORL1","6":"p.R1183P"},{"1":"MB-0739","2":"TL-18-7E6117","3":"0.70","4":"0.8610","5":"TP53","6":"p.R248Q"},{"1":"MB-0754","2":"TL-18-426CA0","3":"0.90","4":"0.8083","5":"NCOR1","6":"p.R1794*"},{"1":"MB-1012","2":"TL-18-59A2F3","3":"0.50","4":"0.6197","5":"TP53","6":"p.P278S"},{"1":"MB-1015","2":"TL-18-79C791","3":"0.80","4":"0.5353","5":"TP53","6":"p.R273H"},{"1":"MB-1018","2":"TL-18-DEF0F4","3":"0.80","4":"0.6475","5":"TP53","6":"p.R273H"},{"1":"MB-1027","2":"TL-18-D474FD","3":"0.90","4":"0.6992","5":"MEN1","6":"p.VV161VF"},{"1":"MB-1029","2":"TL-18-CB87F9","3":"0.80","4":"0.5390","5":"PDGFRA","6":"p.N870S"},{"1":"MB-1035","2":"TL-18-96F651","3":"0.80","4":"0.5170","5":"PIK3CA","6":"p.E542K"},{"1":"MB-1047","2":"TL-18-B0A467","3":"0.80","4":"0.7314","5":"TP53","6":"p.T118fs"},{"1":"MB-1047","2":"TL-18-B0A467","3":"0.80","4":"0.6441","5":"NCOR2","6":"p.M1?"},{"1":"MB-1047","2":"TL-18-B0A467","3":"0.80","4":"0.7198","5":"FANCI","6":"p.Q1122E"},{"1":"MB-1051","2":"TL-18-22E048","3":"0.80","4":"0.6500","5":"TP53","6":"p.H193R"},{"1":"MB-1057","2":"TL-18-461934","3":"0.20","4":"0.5906","5":"TP53","6":"p.P250L"},{"1":"MB-1059","2":"TL-18-8635BE","3":"0.70","4":"0.6700","5":"TP53","6":"p.R213*"},{"1":"MB-1064","2":"TL-18-2A1B1D","3":"0.80","4":"0.7386","5":"TP53","6":"p.R342*"},{"1":"MB-1065","2":"TL-18-D808DB","3":"0.80","4":"0.7590","5":"TP53","6":"p.A159P"},{"1":"MB-1068","2":"TL-18-00ED8C","3":"0.60","4":"0.5460","5":"CASP8","6":"p.M163I"},{"1":"MB-1074","2":"TL-18-93D6FB","3":"0.60","4":"0.7620","5":"FLG","6":"p.N132S"},{"1":"MB-1074","2":"TL-18-93D6FB","3":"0.60","4":"0.5740","5":"PIK3CA","6":"p.E545K"},{"1":"MB-1074","2":"TL-18-93D6FB","3":"0.60","4":"0.7300","5":"TP53","6":"p.R342fs"},{"1":"MB-1077","2":"TL-18-F34FAB","3":"0.90","4":"0.9317","5":"TP53","6":"p.G325*"},{"1":"MB-1077","2":"TL-18-F34FAB","3":"0.90","4":"0.6085","5":"LRP1B","6":"p.T1568A"},{"1":"MB-1085","2":"TL-18-A24B2C","3":"0.80","4":"0.5385","5":"RNF43","6":"p.A115V"},{"1":"MB-1087","2":"TL-18-AE3A66","3":"0.80","4":"0.6193","5":"TP53","6":"p.R342P"},{"1":"MB-1087","2":"TL-18-AE3A66","3":"0.80","4":"0.5106","5":"RPTOR","6":""},{"1":"MB-1088","2":"TL-18-D2B8EF","3":"0.60","4":"0.5780","5":"TP53","6":"p.C176Y"},{"1":"PN0129","2":"TL-18-2F1FB7","3":"0.90","4":"0.6090","5":"SMAD4","6":"p.D537G"},{"1":"PN0142","2":"TL-18-97B262","3":"0.20","4":"1.0000","5":"SMAD4","6":"p.R361C"},{"1":"PN0147","2":"TL-18-213530","3":"0.60","4":"0.5430","5":"CHURC1-FNTB","6":"p.G430S"},{"1":"PN0147","2":"TL-18-213530","3":"0.60","4":"0.7540","5":"TP53","6":"p.R175H"},{"1":"PN0165","2":"TL-18-96D26F","3":"0.80","4":"0.7430","5":"KIF1B","6":"p.R1097C"},{"1":"PN0165","2":"TL-18-96D26F","3":"0.80","4":"0.8290","5":"ROS1","6":"p.Y858*"},{"1":"PN0165","2":"TL-18-96D26F","3":"0.80","4":"0.7330","5":"SMARCB1","6":"p.R374Q"},{"1":"PN0165","2":"TL-18-96D26F","3":"0.80","4":"0.7670","5":"NCOR2","6":"p.P687L"},{"1":"PN0176","2":"TL-18-D90285","3":"0.70","4":"0.5680","5":"PLCG2","6":"p.P734L"},{"1":"PN0177","2":"TL-18-DF6845","3":"0.90","4":"0.7500","5":"DAXX","6":"p.G430D"},{"1":"PN0181","2":"TL-18-2F886A","3":"0.60","4":"0.5190","5":"LZTR1","6":""},{"1":"PN0181","2":"TL-18-2F886A","3":"0.60","4":"0.5080","5":"EP300","6":"p.R2185G"},{"1":"PN0181","2":"TL-18-2F886A","3":"0.60","4":"0.5160","5":"TP53","6":"p.R110fs"},{"1":"PN0189","2":"TL-18-725666","3":"0.90","4":"0.5610","5":"PBRM1","6":"p.L1205L"},{"1":"PN0210","2":"TL-18-3E5182","3":"0.80","4":"0.5870","5":"GFRA2","6":"p.A228V"},{"1":"PN0212","2":"TL-18-2C3DA8","3":"0.85","4":"0.8850","5":"ATP7B","6":"p.E396K"},{"1":"PN0212","2":"TL-18-2C3DA8","3":"0.85","4":"0.7950","5":"CDH1","6":"p.V55M"},{"1":"PN0212","2":"TL-18-2C3DA8","3":"0.85","4":"0.8030","5":"HSPH1","6":""},{"1":"PN0212","2":"TL-18-2C3DA8","3":"0.85","4":"0.5270","5":"POLE","6":"p.S38C"},{"1":"PN0213","2":"TL-18-F9EF49","3":"0.95","4":"0.5580","5":"TP53","6":"p.Q136E"},{"1":"PN0216","2":"TL-18-B45919","3":"0.85","4":"0.5500","5":"ZNRF3","6":"p.P487S"},{"1":"PN0226","2":"TL-18-8C551D","3":"0.90","4":"0.7300","5":"TP53","6":"p.R306*"},{"1":"PN0226","2":"TL-18-8C551D","3":"0.90","4":"0.6110","5":"BRCA1","6":"p.S1569fs"},{"1":"PN0226","2":"TL-18-8C551D","3":"0.90","4":"0.8800","5":"ERBB2(HER2)","6":"p.R1048C"},{"1":"PN0228","2":"TL-18-8DCCAE","3":"0.60","4":"0.7940","5":"NCOR1","6":"p.T525fs"},{"1":"PN0244","2":"TL-18-F1A922","3":"0.80","4":"0.7430","5":"TP53","6":"p.R248W"},{"1":"PN0255","2":"TL-18-5E23D5","3":"0.85","4":"0.5190","5":"TCF3","6":"p.P633L"},{"1":"PN0255","2":"TL-18-5E23D5","3":"0.85","4":"0.5280","5":"VHL","6":"p.E199A"},{"1":"PN0255","2":"TL-18-5E23D5","3":"0.85","4":"0.5280","5":"TNFRSF9","6":"p.R134Q"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<pre class="r"><code>tempusoncoprint &lt;- dplyr::select(tempusoncoprint, -c(PanelName, Normalspecimen, Normalcollecteddate, Normalreceiveddate, Tumorspecimen, Tumorcollecteddate, Tumorreceiveddate, TumorPercentage, TumorMutationalBurden, PercentileTumorMutationalBurd, MicrosatelliteInstabilityStatu, DateSigned, biologicalmeaning, variant_name_C, variant_name_P, NM_number, typeofvariant, VAF, COSMICCases, variantcount, Notes, genecount, alteration))

#cbioportaloncoprint &lt;- tempusoncoprint
#cbioportaloncoprint &lt;- cbioportaloncoprint %&gt;% select(AccessionNo, everything())

#cbioportaloncoprint$Gene &lt;- as.character(cbioportaloncoprint$Gene)
#cbioportaloncoprint$Gene &lt;- ifelse (cbioportaloncoprint$Gene==&quot;ERBB2&quot; | cbioportaloncoprint$Gene==&quot;ERBB2 (HER2)&quot; | cbioportaloncoprint$Gene==&quot;ERBB2(HER2)&quot; | cbioportaloncoprint$Gene==&quot;ERBB2HER2)&quot; | cbioportaloncoprint$Gene==&quot;HER2&quot;, &quot;ERBB2/HER2&quot;, cbioportaloncoprint$Gene)

#cbioportaloncoprint &lt;- cbioportaloncoprint %&gt;% dplyr::filter(Gene==&quot;TP53&quot; | Gene==&quot;MCL1&quot; | Gene==&quot;PIK3CA&quot; | Gene==&quot;ERBB2/HER2&quot; | Gene==&quot;ZFHX3&quot; | Gene==&quot;ATM&quot; | Gene==&quot;BCORL1&quot; | Gene==&quot;FAT1&quot; | Gene==&quot;KMT2B&quot; | Gene==&quot;KMT2D&quot; | Gene==&quot;NOTCH1&quot; | Gene==&quot;AMER1&quot; | Gene==&quot;CDK12&quot; | Gene==&quot;KDM5C&quot; | Gene==&quot;FLG&quot; | Gene==&quot;MYC&quot; | Gene==&quot;NOTCH3&quot; | Gene==&quot;TSC2&quot; | Gene==&quot;ALK&quot; | Gene==&quot;CUX1&quot; | Gene==&quot;GATA3&quot; | Gene==&quot;HNF1B&quot; | Gene==&quot;SPEN&quot; | Gene==&quot;ARID1A&quot; | Gene==&quot;BCL11B&quot; | Gene==&quot;BCOR&quot; | Gene==&quot;CIC&quot; | Gene==&quot;CDH1&quot; | Gene==&quot;ATRX&quot; | Gene==&quot;BRCA1&quot; | Gene==&quot;BRCA2&quot;)

#cbioportaloncoprint$X&lt;-NULL
#cbioportaloncoprint$StudyID&lt;-NULL
#cbioportaloncoprint$caseID&lt;-NULL

#write.table(cbioportaloncoprint, &quot;cbioportaloncoprint.tsv&quot;, sep=&quot;\t&quot;, row.names=FALSE, col.names=FALSE)

tempusoncoprint &lt;- tempusoncoprint %&gt;% spread(Gene,variant)

tempusoncoprint$V1=NULL #Clears artificial columns that were created in the process of formatting the dataset
tempusoncoprint$X=NULL

tempusoncoprint$`ERBB2/HER2` &lt;- coalesce(tempusoncoprint$ERBB2, tempusoncoprint$`ERBB2 (HER2)`, tempusoncoprint$`ERBB2(HER2)`, tempusoncoprint$`ERBB2HER2)`, tempusoncoprint$HER2)

tempusoncoprint$`KMT2A/MLL` &lt;- coalesce(tempusoncoprint$MLL1, tempusoncoprint$`KMT2A(MLL)`, tempusoncoprint$`KMT2A (MLL)`, tempusoncoprint$`KMT2A (MLL1)`)

tempusoncoprint$`KMT2C/MLL3` &lt;- coalesce(tempusoncoprint$MLL3, tempusoncoprint$`KMT2C(MLL3)`, tempusoncoprint$`KMT2C (MLL3)`)

tempusoncoprint &lt;- tempusoncoprint %&gt;% dplyr::select(caseID, subtype, TP53, MCL1, PIK3CA, `ERBB2/HER2`, ZFHX3, ATM, BCORL1, FAT1, KMT2B, KMT2D, NOTCH1, AMER1, CDK12, KDM5C, FLG, MYC, NOTCH3, TSC2, ALK, CUX1, GATA3, HNF1B, SPEN, ARID1A, BCL11B, BCOR, CIC, CDH1, `KMT2C/MLL3`, ATRX, BRCA1, BRCA2, CDK12, KDM6A, BCR, CHD4, CIC, EGFR, IRS2, JUN, KAT6A, LDLR, MEF2B, MSH6, MUTYH, DNMT3A, ETV6, LRP1B, MSH2, PTEN, TSC2, AR, RARA, MDM2, CTCF)

tempusoncoprintHRposHERneg &lt;- tempusoncoprint %&gt;% filter(subtype==&quot;HER-/HR+&quot;)
tempusoncoprintHRposHERneg$StudyID=NULL
tempusoncoprintHRposHERneg$AccessionNo=NULL
tempusoncoprintHRposHERneg$subtype=NULL

mat = as.matrix(tempusoncoprintHRposHERneg)
mat[is.na(mat)] = &quot;&quot;
rownames(mat) = mat[, 1]
mat = mat[, -1]
mat=  mat[, -ncol(mat)]
mat = t(mat)

col = c(&quot;CNAGAIN&quot; = &quot;#009999&quot;, &quot;CNALOSS&quot; = &quot;#CC0066&quot;, &quot;INFRAME&quot; = &quot;#FF8000&quot;, &quot;MISSENSE&quot; = &quot;#0066CC&quot;, &quot;PROMOTER&quot; = &quot;#EFD915&quot;, &quot;TRUNC&quot; = &quot;black&quot;, &quot;OTHER&quot; = &quot;#006600&quot;)

alter_fun = list(
    CNAGAIN = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNAGAIN&quot;], col = NA)),
    CNALOSS = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNALOSS&quot;], col = NA)),
    INFRAME = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;INFRAME&quot;], col = NA)),
    MISSENSE = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;MISSENSE&quot;], col = NA)),
    PROMOTER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;PROMOTER&quot;], col = NA)),
    TRUNC = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;TRUNC&quot;], col = NA)),
    OTHER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;OTHER&quot;], col = NA))
)

column_title = &quot;OncoPrint for University of Chicago samples: HR+, HER2-&quot;
heatmap_legend_param = list(title = &quot;Alterations&quot;, at = c(&quot;CNAGAIN&quot;,&quot;CNALOSS&quot;, &quot;INFRAME&quot;, &quot;MISSENSE&quot;, &quot;PROMOTER&quot;, &quot;TRUNC&quot;, &quot;OTHER&quot;), labels = c(&quot;Copy number gain&quot;, &quot;Copy number loss&quot;,&quot;Insertion/Deletion&quot;, &quot;Missense&quot;, &quot;Splice mutation&quot;, &quot;Nonsense&quot;, &quot;Mutation-Other&quot;))
oncoPrint(mat,
    alter_fun = alter_fun,
    col = col, 
    column_title = column_title,
    heatmap_legend_param = heatmap_legend_param,
    show_pct=FALSE,
    remove_empty_columns = TRUE,
    row_names_gp = gpar(fontsize = 10)
)</code></pre>
<p><img src="figure/Tempus.Rmd/Oncoprint-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusoncoprintHRposHERpos &lt;- tempusoncoprint %&gt;% filter(subtype==&quot;HER+/HR+&quot;)
tempusoncoprintHRposHERpos$StudyID=NULL
tempusoncoprintHRposHERpos$AccessionNo=NULL
tempusoncoprintHRposHERpos$subtype=NULL

mat = as.matrix(tempusoncoprintHRposHERpos)
mat[is.na(mat)] = &quot;&quot;
rownames(mat) = mat[, 1]
mat = mat[, -1]
mat=  mat[, -ncol(mat)]
mat = t(mat)

col = c(&quot;CNAGAIN&quot; = &quot;#009999&quot;, &quot;CNALOSS&quot; = &quot;#CC0066&quot;, &quot;INFRAME&quot; = &quot;#FF8000&quot;, &quot;MISSENSE&quot; = &quot;#0066CC&quot;, &quot;PROMOTER&quot; = &quot;#EFD915&quot;, &quot;TRUNC&quot; = &quot;black&quot;, &quot;OTHER&quot; = &quot;#006600&quot;)

alter_fun = list(
    CNAGAIN = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNAGAIN&quot;], col = NA)),
    CNALOSS = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNALOSS&quot;], col = NA)),
    INFRAME = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;INFRAME&quot;], col = NA)),
    MISSENSE = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;MISSENSE&quot;], col = NA)),
    PROMOTER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;PROMOTER&quot;], col = NA)),
    TRUNC = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;TRUNC&quot;], col = NA)),
    OTHER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;OTHER&quot;], col = NA))
)

column_title = &quot;OncoPrint for University of Chicago samples: HR+, HER2+&quot;
heatmap_legend_param = list(title = &quot;Alterations&quot;, at = c(&quot;CNAGAIN&quot;,&quot;CNALOSS&quot;, &quot;INFRAME&quot;, &quot;MISSENSE&quot;, &quot;PROMOTER&quot;, &quot;TRUNC&quot;, &quot;OTHER&quot;), labels = c(&quot;Copy number gain&quot;, &quot;Copy number loss&quot;,&quot;Insertion/Deletion&quot;, &quot;Missense&quot;, &quot;Splice mutation&quot;, &quot;Nonsense&quot;, &quot;Mutation-Other&quot;))
oncoPrint(mat,
    alter_fun = alter_fun,
    col = col, 
    column_title = column_title,
    heatmap_legend_param = heatmap_legend_param,
    show_pct=FALSE,
    remove_empty_columns = TRUE,
    row_names_gp = gpar(fontsize = 10)
)</code></pre>
<p><img src="figure/Tempus.Rmd/Oncoprint-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusoncoprintHERpos &lt;- tempusoncoprint %&gt;% filter(subtype==&quot;HER+/HR-&quot;)
tempusoncoprintHERpos$StudyID=NULL
tempusoncoprintHERpos$AccessionNo=NULL
tempusoncoprintHERpos$subtype=NULL

mat = as.matrix(tempusoncoprintHERpos)
mat[is.na(mat)] = &quot;&quot;
rownames(mat) = mat[, 1]
mat = mat[, -1]
mat=  mat[, -ncol(mat)]
mat = t(mat)

col = c(&quot;CNAGAIN&quot; = &quot;#009999&quot;, &quot;CNALOSS&quot; = &quot;#CC0066&quot;, &quot;INFRAME&quot; = &quot;#FF8000&quot;, &quot;MISSENSE&quot; = &quot;#0066CC&quot;, &quot;PROMOTER&quot; = &quot;#EFD915&quot;, &quot;TRUNC&quot; = &quot;black&quot;, &quot;OTHER&quot; = &quot;#006600&quot;)

alter_fun = list(
    CNAGAIN = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNAGAIN&quot;], col = NA)),
    CNALOSS = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNALOSS&quot;], col = NA)),
    INFRAME = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;INFRAME&quot;], col = NA)),
    MISSENSE = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;MISSENSE&quot;], col = NA)),
    PROMOTER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;PROMOTER&quot;], col = NA)),
    TRUNC = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;TRUNC&quot;], col = NA)),
    OTHER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;OTHER&quot;], col = NA))
)

column_title = &quot;OncoPrint for University of Chicago samples: HR-, HER2+&quot;
heatmap_legend_param = list(title = &quot;Alterations&quot;, at = c(&quot;CNAGAIN&quot;,&quot;CNALOSS&quot;, &quot;INFRAME&quot;, &quot;MISSENSE&quot;, &quot;PROMOTER&quot;, &quot;TRUNC&quot;, &quot;OTHER&quot;), labels = c(&quot;Copy number gain&quot;, &quot;Copy number loss&quot;,&quot;Insertion/Deletion&quot;, &quot;Missense&quot;, &quot;Splice mutation&quot;, &quot;Nonsense&quot;, &quot;Mutation-Other&quot;))
oncoPrint(mat,
    alter_fun = alter_fun,
    col = col, 
    column_title = column_title,
    heatmap_legend_param = heatmap_legend_param,
    show_pct=FALSE,
    remove_empty_columns = TRUE,
    row_names_gp = gpar(fontsize = 10)
)</code></pre>
<p><img src="figure/Tempus.Rmd/Oncoprint-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusoncoprintTNBC &lt;- tempusoncoprint %&gt;% filter(subtype==&quot;TNBC&quot;)
tempusoncoprintTNBC$StudyID=NULL
tempusoncoprintTNBC$AccessionNo=NULL
tempusoncoprintTNBC$subtype=NULL

mat = as.matrix(tempusoncoprintTNBC)
mat[is.na(mat)] = &quot;&quot;
rownames(mat) = mat[, 1]
mat = mat[, -1]
mat=  mat[, -ncol(mat)]
mat = t(mat)

col = c(&quot;CNAGAIN&quot; = &quot;#009999&quot;, &quot;CNALOSS&quot; = &quot;#CC0066&quot;, &quot;INFRAME&quot; = &quot;#FF8000&quot;, &quot;MISSENSE&quot; = &quot;#0066CC&quot;, &quot;PROMOTER&quot; = &quot;#EFD915&quot;, &quot;TRUNC&quot; = &quot;black&quot;, &quot;OTHER&quot; = &quot;#006600&quot;)

alter_fun = list(
    CNAGAIN = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNAGAIN&quot;], col = NA)),
    CNALOSS = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;CNALOSS&quot;], col = NA)),
    INFRAME = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;INFRAME&quot;], col = NA)),
    MISSENSE = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;MISSENSE&quot;], col = NA)),
    PROMOTER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;PROMOTER&quot;], col = NA)),
    TRUNC = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;TRUNC&quot;], col = NA)),
    OTHER = function(x, y, w, h)
        grid.rect(x, y, w*1, h*1, 
            gp = gpar(fill = col[&quot;OTHER&quot;], col = NA))
)

column_title = &quot;OncoPrint for University of Chicago samples: TNBC&quot;
heatmap_legend_param = list(title = &quot;Alterations&quot;, at = c(&quot;CNAGAIN&quot;,&quot;CNALOSS&quot;, &quot;INFRAME&quot;, &quot;MISSENSE&quot;, &quot;PROMOTER&quot;, &quot;TRUNC&quot;, &quot;OTHER&quot;), labels = c(&quot;Copy number gain&quot;, &quot;Copy number loss&quot;,&quot;Insertion/Deletion&quot;, &quot;Missense&quot;, &quot;Splice mutation&quot;, &quot;Nonsense&quot;, &quot;Mutation-Other&quot;))
oncoPrint(mat,
    alter_fun = alter_fun,
    col = col, 
    column_title = column_title,
    heatmap_legend_param = heatmap_legend_param,
    show_pct=FALSE,
    remove_empty_columns = TRUE,
    row_names_gp = gpar(fontsize = 10)
)</code></pre>
<p><img src="figure/Tempus.Rmd/Oncoprint-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  0.000   1.300   2.900   7.562   6.050  76.300 </code></pre>
<p><img src="figure/Tempus.Rmd/Ongoing%20mutational%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Ongoing mutational summary-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
7fbf15a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
<tr>
<td style="text-align:left;">
99339fe
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-10
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>Warning in chisq.test(tempusmergedupclean$BRCAYN, tempusmergedupclean$TMB):
Chi-squared approximation may be incorrect</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  tempusmergedupclean$BRCAYN and tempusmergedupclean$TMB
X-squared = 71.777, df = 37, p-value = 0.0005269</code></pre>
<p>Only 88 of 139 reports contained percentile information; this was not included for analysis. Tumor mutational burden was measured in m/MB.</p>
<p>Increased TMB was associated with BRCA1 and 2 mutational status.</p>
<table>
<caption>Distribution of MSI status</caption>
<thead>
<tr class="header">
<th align="left">MSI status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Equivocal</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">High</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">MSI can not be assessed because the tumor content is below 30%.</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">Stable</td>
<td align="right">64</td>
</tr>
<tr class="odd">
<td align="left">Stable (MSS)</td>
<td align="right">43</td>
</tr>
<tr class="even">
<td align="left">Undetermined</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td align="left">The majority of patients had stable MSI status in their tumors; on</td>
<td align="right">ly 1.5% showed MSI unstable disease.</td>
</tr>
</tbody>
</table>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.00000 0.06972 0.10595 0.11215 0.15008 0.30414      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
 0.1271  0.2409  0.2714  0.2718  0.3018  0.3944      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.03134 0.09566 0.13081 0.12943 0.16956 0.26398      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-3.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-3.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
 0.2139  0.3529  0.3711  0.3835  0.4171  0.5216      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-4.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-4.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.04802 0.06850 0.09240 0.10309 0.12424 0.22419      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-5.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-5.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
0.00000 0.05514 0.10400 0.11939 0.16116 0.82456      58 </code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20summary-6.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune summary-6.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>74 patients had “immune profile” data as calculated by Tempus. This represents a derivation from the RNA-seq data that was developed from a machine learning algorithm, NOT a pathology-based assessment. The immune score is a “support vector regression model for the immune cell estimates and formal thresholds for high or low are not in place as of yet.”</p>
<p>Study IDs MB-0910 and MB-1073 represent patients with metastatic disease. Going forward, we will exclude these patients from the cohort to better gauge the effect of our markers under evaluation on survival and recurrence.</p>
</div>
<div id="recurrence-and-survival-by-ethnicity-in-our-cohort-significant" class="section level1">
<h1>Recurrence and survival by ethnicity in our cohort (significant)</h1>
<pre class="r"><code>tempusmergedupclean &lt;- tempusmergedupclean %&gt;% filter(stage14!=4)
  
fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$bw, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by ethnicity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Ethnicity%20recurrence%20and%20survival-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Ethnicity recurrence and survival-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$bw, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Progression-free survival in patients by ethnicity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Ethnicity%20recurrence%20and%20survival-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Ethnicity recurrence and survival-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="mutations-among-different-subgroups---subtype-and-race-tp53-significant-in-general" class="section level1">
<h1>Mutations among different subgroups - Subtype and race (TP53 significant in general)</h1>
<pre class="r"><code>subtypeHERposHRneg &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER+/HR-&quot;)
allgenesHERposHRneg &lt;- colSums(!((subtypeHERposHRneg[,105:663])==&quot;&quot;))
allgenesHERposHRneg[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERposHRneg[&quot;ERBB2&quot;], allgenesHERposHRneg[&quot;ERBB2 (HER2)&quot;], allgenesHERposHRneg[&quot;ERBB2(HER2)&quot;], allgenesHERposHRneg[&quot;ERBB2HER2)&quot;], allgenesHERposHRneg[&quot;HER2&quot;])
headHERposHRneg &lt;- head(sort(allgenesHERposHRneg, decreasing=TRUE), 20)
headHERposHRneg</code></pre>
<pre><code>        TP53        CDK12        ERBB2       PIK3CA ERBB2 (HER2) 
           9            6            6            4            3 
         FLG        KDM6A         MCL1         BCOR          BCR 
           3            3            3            2            2 
        CHD4          CIC         EGFR         IRS2          JUN 
           2            2            2            2            2 
       KAT6A         LDLR        MEF2B         MSH6        MUTYH 
           2            2            2            2            2 </code></pre>
<pre class="r"><code>subtypeHERnegHRpos &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER-/HR+&quot;)
allgenesHERnegHRpos &lt;- colSums(!((subtypeHERnegHRpos[,105:663])==&quot;&quot;))
allgenesHERnegHRpos[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERnegHRpos[&quot;ERBB2&quot;], allgenesHERnegHRpos[&quot;ERBB2 (HER2)&quot;], allgenesHERnegHRpos[&quot;ERBB2(HER2)&quot;], allgenesHERnegHRpos[&quot;ERBB2HER2)&quot;], allgenesHERnegHRpos[&quot;HER2&quot;])
headHERnegHRpos &lt;- head(sort(allgenesHERnegHRpos, decreasing=TRUE), 20)
headHERnegHRpos</code></pre>
<pre><code>  MCL1 PIK3CA   TP53  ZFHX3 BCORL1  KMT2B  KMT2D    ALK  AMER1   FAT1 
    22     20     19     15     14     13     12     11     11     11 
 KDM5C  LZTR1 NOTCH1    ATM MAP3K1 NOTCH3   SPEN BCL11B   CDH1    CIC 
    11     11     11     10     10     10     10      9      9      9 </code></pre>
<pre class="r"><code>subtypeTNBC &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;TNBC&quot;)
allgenesTNBC &lt;- colSums(!((subtypeTNBC[,105:663])==&quot;&quot;))
allgenesTNBC[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesTNBC[&quot;ERBB2&quot;], allgenesTNBC[&quot;ERBB2 (HER2)&quot;], allgenesTNBC[&quot;ERBB2(HER2)&quot;], allgenesTNBC[&quot;ERBB2HER2)&quot;], allgenesTNBC[&quot;HER2&quot;])
headTNBC&lt;- head(sort(allgenesTNBC, decreasing=TRUE), 20)
headTNBC</code></pre>
<pre><code>  TP53   MCL1 NOTCH1    FLG  KMT2C    MYC  AMER1   CDK4 CDKN1B DNMT3A 
    25      8      7      5      5      5      4      4      4      4 
  ETV6   FAT1   KRAS  LRP1B   MSH2   PTEN   TSC2  ZFHX3    ALK     AR 
     4      4      4      4      4      4      4      4      3      3 </code></pre>
<pre class="r"><code>subtypeHERposHRpos &lt;- tempusmergedupclean %&gt;% filter(subtype==&quot;HER+/HR+&quot;)
allgenesHERposHRpos &lt;- colSums(!((subtypeHERposHRpos[,105:663])==&quot;&quot;))
allgenesHERposHRpos[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesHERposHRpos[&quot;ERBB2&quot;], allgenesHERposHRpos[&quot;ERBB2 (HER2)&quot;], allgenesHERposHRpos[&quot;ERBB2(HER2)&quot;], allgenesHERposHRpos[&quot;ERBB2HER2)&quot;], allgenesHERposHRpos[&quot;HER2&quot;])
headHERposHRpos &lt;- head(sort(allgenesHERposHRpos, decreasing=TRUE), 20)
headHERposHRpos</code></pre>
<pre><code>        TP53       PIK3CA ERBB2 (HER2)          ATM         MCL1 
          17           10            9            8            8 
       CDK12         RARA        AMER1        HNF1B         MDM2 
           7            7            6            6            6 
      BCL11B         CDK4          FLG        GATA3          MYC 
           5            5            5            5            5 
        NRG1       BCORL1        BRCA1         CDH1         CTCF 
           5            4            4            4            4 </code></pre>
<pre class="r"><code>chisq.test(headHERnegHRpos, headHERposHRneg, headTNBC, headHERposHRpos)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  headHERnegHRpos and headHERposHRneg
X-squared = 74.444, df = 36, p-value = 0.0001725</code></pre>
<pre class="r"><code>subtypewhite &lt;- tempusmergedupclean %&gt;% filter(race==&quot;White&quot;)
allgeneswhite&lt;- colSums(!((subtypewhite[,105:663])==&quot;&quot;))
allgeneswhite[&quot;ERBB2/HER2&quot;] &lt;- sum(allgeneswhite[&quot;ERBB2&quot;], allgeneswhite[&quot;ERBB2 (HER2)&quot;], allgeneswhite[&quot;ERBB2(HER2)&quot;], allgeneswhite[&quot;ERBB2HER2)&quot;], allgeneswhite[&quot;HER2&quot;])
headwhite &lt;- head(sort(allgeneswhite, decreasing=TRUE), 5)
headwhite</code></pre>
<pre><code>  TP53   MCL1 PIK3CA BCORL1  KDM5C 
    32     25     19     13     13 </code></pre>
<pre class="r"><code>subtypeblack &lt;- tempusmergedupclean %&gt;% filter(race==&quot;Black&quot;)
allgenesblack&lt;- colSums(!((subtypeblack[,105:663])==&quot;&quot;))
allgenesblack[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesblack[&quot;ERBB2&quot;], allgenesblack[&quot;ERBB2 (HER2)&quot;], allgenesblack[&quot;ERBB2(HER2)&quot;], allgenesblack[&quot;ERBB2HER2)&quot;], allgenesblack[&quot;HER2&quot;])
headblack &lt;- head(sort(allgenesblack, decreasing=TRUE), 5)
headblack</code></pre>
<pre><code>  TP53 PIK3CA   MCL1  ZFHX3    ATM 
    40     19     18     15     12 </code></pre>
<pre class="r"><code>chisq.test(headwhite, headblack)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  headwhite and headblack
X-squared = 15, df = 12, p-value = 0.2414</code></pre>
<p>We saw a trend towards significance in our cohort between breast cancer subtype and the most commonly mutated genes (outside of overrepresentation of ERBB2/HER2 in the HER2+ subtype), as well as no significant difference between ethnicty and most commonly mutated genes.</p>
<pre class="r"><code>#TP53 status
tempusmergedupclean$TP53YN &lt;- ifelse (tempusmergedupclean$TP53==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$TP53YN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$TP53YN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>#BRCA status
fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$BRCAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by BRCA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-3.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-3.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$BRCAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by BRCA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-4.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-4.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>#AR receptor status
tempusmergedupclean$ARYN &lt;- ifelse (tempusmergedupclean$AR==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$ARYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by AR mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-5.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-5.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$ARYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by AR mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-6.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-6.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>#PIK3CA mutation status
tempusmergedupclean$PIK3CAYN &lt;- ifelse (tempusmergedupclean$PIK3CA==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$PIK3CAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by PIK3CA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-7.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-7.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$PIK3CAYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by PIK3CA mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-8.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-8.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>#ATM mutation status
tempusmergedupclean$ATMYN &lt;- ifelse (tempusmergedupclean$ATM==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$ATMYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by ATM mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-9.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-9.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$ATMYN, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by ATM mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Outcomes%20by%20mutation%20status-10.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Outcomes by mutation status-10.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="immune-profile-data-and-association-with-survival-and-recurrence-tmb-trends-with-recurrence" class="section level1">
<h1>Immune profile data and association with survival and recurrence (TMB trends with recurrence)</h1>
<pre class="r"><code>chisq.test (tempusmergedupclean$neoadj, tempusmergedupclean$TMB)</code></pre>
<pre><code>Warning in chisq.test(tempusmergedupclean$neoadj, tempusmergedupclean$TMB):
Chi-squared approximation may be incorrect</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  tempusmergedupclean$neoadj and tempusmergedupclean$TMB
X-squared = 27.692, df = 27, p-value = 0.427</code></pre>
<pre class="r"><code>tempusmergedupclean$TMBhighlow &lt;- ifelse(tempusmergedupclean$TMB&gt;9, 1, 0)
  
fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$TMBhighlow, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by TMB mutation status (high&gt;9 m/MB)&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$TMBhighlow, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Progression-free survival in patients by TMB (high&gt;9 m/mB)&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$TMBquartile &lt;- tempusmergedupclean$TMB
tempusmergedupclean$TMBquartile &lt;- ifelse(tempusmergedupclean$TMBquartile&lt;1.3, 1, tempusmergedupclean$TMBquartile)
tempusmergedupclean$TMBquartile &lt;- ifelse(tempusmergedupclean$TMBquartile&gt;=1.3 &amp; tempusmergedupclean$TMBquartile&lt;2.9, 2, tempusmergedupclean$TMBquartile)
tempusmergedupclean$TMBquartile &lt;- ifelse(tempusmergedupclean$TMBquartile&gt;=2.9 &amp; tempusmergedupclean$TMBquartile&lt;6.05, 3, tempusmergedupclean$TMBquartile)
tempusmergedupclean$TMBquartile &lt;- ifelse(tempusmergedupclean$TMBquartile&gt;=6.05, 4, tempusmergedupclean$TMBquartile)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$TMBquartile, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by TMB mutation status (quartile)&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-3.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-3.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$TMBquartile, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by TMB quartile&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-4.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-4.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>chisq.test(tempusmergedupclean$stage14, tempusmergedupclean$TMBhighlow)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test

data:  tempusmergedupclean$stage14 and tempusmergedupclean$TMBhighlow
X-squared = 7.1218, df = 2, p-value = 0.02841</code></pre>
<pre class="r"><code>tempusmergedupclean$immunemedian &lt;- ifelse(tempusmergedupclean$immune&gt;0.12, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$immunemedian, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by immune score greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-5.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-5.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$immunemedian, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by immune score greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-6.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-6.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$bmedian &lt;- ifelse(tempusmergedupclean$b&gt;0.10, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$bmedian, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by % b cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-7.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-7.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$bmedian, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by % b cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-8.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-8.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$cd4median &lt;- ifelse(tempusmergedupclean$cd4&gt;0.27, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$cd4median, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by % cd4 cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-9.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-9.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$cd4median, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by % cd4 cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-10.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-10.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$cd8median &lt;- ifelse(tempusmergedupclean$cd8&gt;0.13, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$cd8median, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by % cd8 cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-11.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-11.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$cd8median, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by % cd8 cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-12.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-12.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$macmedian &lt;- ifelse(tempusmergedupclean$mac&gt;0.37, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$macmedian, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by % macrophage cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-13.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-13.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$macmedian, 
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by % macrophage infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-14.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-14.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>tempusmergedupclean$nkmedian &lt;- ifelse(tempusmergedupclean$nk&gt;0.092, 1, 0)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$nkmedian, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by % NK cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-15.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-15.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusmergedupclean$pfstimefu, as.numeric(tempusmergedupclean$pfs)) ~tempusmergedupclean$nkmedian,
               data=tempusmergedupclean)
ggsurvplot(fitrecurrence, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in patients by % NK cell infiltrate greater or less than the median&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-16.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-16.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>surv_object &lt;- Surv(time=tempusmergedupclean$timefu, event=as.numeric(tempusmergedupclean$vitalstatussurv))
     
fit.coxph &lt;- coxph(surv_object ~ tempusmergedupclean$TMBhighlow + tempusmergedupclean$immunemedian + tempusmergedupclean$bmedian + tempusmergedupclean$cd4median + tempusmergedupclean$cd8median + tempusmergedupclean$macmedian + tempusmergedupclean$nkmedian,
                   data = tempusmergedupclean)
                   
ggforest(fit.coxph, data = tempusmergedupclean, main=&quot;Comparative hazard ratios of immune measurements in patient tumors&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-17.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-17.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>rec_object &lt;- Surv(time=tempusmergedupclean$pfstimefu, event=as.numeric(tempusmergedupclean$pfs))
     
fit.coxph2 &lt;- coxph(rec_object ~ tempusmergedupclean$TMBhighlow + tempusmergedupclean$immunemedian + tempusmergedupclean$bmedian + tempusmergedupclean$cd4median + tempusmergedupclean$cd8median + tempusmergedupclean$macmedian + tempusmergedupclean$nkmedian,
                   data = tempusmergedupclean)
                   
ggforest(fit.coxph2, data = tempusmergedupclean, main=&quot;Comparative hazard ratios of immune measurements in patient tumors&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Immune%20profile%20recurrence%20and%20survival-18.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Immune profile recurrence and survival-18.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>Neoadjuvant chemotherapy treatment status was not associated with a change in tumor mutational burden. Based on Chalmers et al (<a href="https://genomemedicine.biomedcentral.com/track/pdf/10.1186/s13073-017-0424-2?site=genomemedicine.biomedcentral.com" class="uri">https://genomemedicine.biomedcentral.com/track/pdf/10.1186/s13073-017-0424-2?site=genomemedicine.biomedcentral.com</a>), we defined a high TMB as &gt;20. We saw a nonsignificant trend towards increased mutational burden and immune infiltrate being associated with reduced likelihood of recurrence in these patients.</p>
<pre class="r"><code>tempusmergedupclean$CDKpathYN &lt;- ifelse (tempusmergedupclean$CDK4==&quot;&quot; &amp; tempusmergedupclean$CCND1==&quot;&quot; &amp; tempusmergedupclean$CDKN2A==&quot;&quot; &amp; tempusmergedupclean$CDK6==&quot;&quot; &amp; tempusmergedupclean$RB1==&quot;&quot; &amp; tempusmergedupclean$FAT1==&quot;&quot;, 0, 1)

subtypeHERnegHRpos$CDKsens &lt;- ifelse(grepl(&quot;gain&quot;, as.character(subtypeHERnegHRpos$CDK4[1:51])), 1, 0)
subtypeHERnegHRpos$CDKsens &lt;- ifelse(grepl(&quot;gain&quot;, as.character(subtypeHERnegHRpos$CCND1[1:51])), 1, subtypeHERnegHRpos$CDKsens)
subtypeHERnegHRpos$CDKsens &lt;- ifelse(grepl(&quot;loss&quot;, as.character(subtypeHERnegHRpos$CDKN2A[1:51])), 1, subtypeHERnegHRpos$CDKsens)

subtypeHERnegHRpos$CDKres &lt;- ifelse(grepl(&quot;loss&quot;, as.character(subtypeHERnegHRpos$RB1[1:51])), 1, 0)
subtypeHERnegHRpos$CDKres &lt;- ifelse(grepl(&quot;gain&quot;, as.character(subtypeHERnegHRpos$CDK6[1:51])), 1, subtypeHERnegHRpos$CDKres)
subtypeHERnegHRpos$CDKres &lt;- ifelse(grepl(&quot;gain&quot;, as.character(subtypeHERnegHRpos$FAT1[1:51])), 1, subtypeHERnegHRpos$CDKres)

fitsurvival &lt;- survfit(Surv(time=subtypeHERnegHRpos$timefu, as.numeric(subtypeHERnegHRpos$vitalstatussurv)) ~subtypeHERnegHRpos$CDKsens, 
               data=subtypeHERnegHRpos)
ggsurvplot(fitsurvival, data=subtypeHERnegHRpos, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of HR+, HER2- patients by CDK mutation sensitivity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/CDK%20pathway%20status-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of CDK pathway status-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=subtypeHERnegHRpos$pfstimefu, as.numeric(subtypeHERnegHRpos$pfs)) ~subtypeHERnegHRpos$CDKsens,
               data=subtypeHERnegHRpos)
ggsurvplot(fitrecurrence, data=subtypeHERnegHRpos, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Progression-free survival of HR+, HER2- patients by CDK mutation sensitivity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/CDK%20pathway%20status-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of CDK pathway status-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitsurvival &lt;- survfit(Surv(time=subtypeHERnegHRpos$timefu, as.numeric(subtypeHERnegHRpos$vitalstatussurv)) ~subtypeHERnegHRpos$CDKres, 
               data=subtypeHERnegHRpos)
ggsurvplot(fitsurvival, data=subtypeHERnegHRpos, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of HR+, HER2- patients by CDK mutation resistance&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/CDK%20pathway%20status-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=subtypeHERnegHRpos$pfstimefu, as.numeric(subtypeHERnegHRpos$pfs)) ~subtypeHERnegHRpos$CDKres,
               data=subtypeHERnegHRpos)
ggsurvplot(fitrecurrence, data=subtypeHERnegHRpos, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Recurrence likelihood of HR+, HER2- patients by CDK mutation sensitivity&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/CDK%20pathway%20status-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>tempusmergedupclean$HRDpathYN &lt;- ifelse (tempusmergedupclean$BRCA1==&quot;&quot; &amp; tempusmergedupclean$BRCA2==&quot;&quot; &amp; tempusmergedupclean$RAD51C==&quot;&quot; &amp; tempusmergedupclean$RAD51D==&quot;&quot; &amp; tempusmergedupclean$RAD50==&quot;&quot; &amp; tempusmergedupclean$BRIP1==&quot;&quot; &amp; tempusmergedupclean$BARD1==&quot;&quot; &amp; tempusmergedupclean$CHEK2==&quot;&quot; &amp; tempusmergedupclean$MRE11==&quot;&quot;  &amp; tempusmergedupclean$PALB2==&quot;&quot; &amp; tempusmergedupclean$MLH1==&quot;&quot; &amp; tempusmergedupclean$MSH2==&quot;&quot;  &amp; tempusmergedupclean$MSH6==&quot;&quot; &amp; tempusmergedupclean$PMS2==&quot;&quot;, 0, 1)

fitsurvival &lt;- survfit(Surv(time=tempusmergedupclean$timefu, as.numeric(tempusmergedupclean$vitalstatussurv)) ~tempusmergedupclean$HRDpathYN, 
               data=tempusmergedupclean)
ggsurvplot(fitsurvival, data=tempusmergedupclean, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of patients by HRD mutation pathway status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/CDK%20pathway%20status-5.png" width="672" style="display: block; margin: auto;" /> Mutations in the CDK pathway are associated with survival (significant) and have a trend towards significance in recurrence in patients with TNBC and HER-, HR+ disease. There was no specific association with ethnicity nor an association with mutational burden/BRCA status.</p>
</div>
<div id="her2-rna-correlation-with-mutation-status" class="section level1">
<h1>HER2 RNA correlation with mutation status</h1>
<pre class="r"><code>FOLDER &lt;- &quot;/Users/parajago/Research-Local/Tempus-Local/RNA&quot;
sampleFiles &lt;- grep(&quot;rna_expression.csv&quot;,list.files(FOLDER),value=TRUE)

sampleFiles &lt;- gsub(&quot;_.rna_expression.csv&quot;,&quot;&quot;,sampleFiles)
sampleFiles &lt;- gsub(&quot;MB-0983_3480666586.rna_expression.csv&quot;,&quot;MB-0983&quot;,sampleFiles)
sampleFiles &lt;- gsub(&quot;MB-1029_8017186.rna_expression.csv&quot;,&quot;MB-1029&quot;,sampleFiles)
sampleFiles &lt;- gsub(&quot;PN0206_PN0206-LN.rna_expression.csv&quot;,&quot;PN0206&quot;,sampleFiles)
sampleFiles &lt;- gsub(&quot;PN0206_PN0206.rna_expression.csv&quot;,&quot;PN0206&quot;,sampleFiles)

sampleFiles</code></pre>
<pre><code> [1] &quot;MB-0138&quot; &quot;MB-0983&quot; &quot;MB-1022&quot; &quot;MB-1029&quot; &quot;MB-1069&quot; &quot;MB-1088&quot; &quot;PN0133&quot; 
 [8] &quot;PN0143&quot;  &quot;PN0174&quot;  &quot;PN0176&quot;  &quot;PN0186&quot;  &quot;PN0187&quot;  &quot;PN0206&quot;  &quot;PN0206&quot; 
[15] &quot;PN0206&quot;  &quot;PN0212&quot;  &quot;PN0213&quot;  &quot;PN0216&quot;  &quot;PN0226&quot;  &quot;PN0232&quot;  &quot;PN0240&quot; 
[22] &quot;PN0255&quot;  &quot;PN0263&quot;  &quot;PN0281&quot;  &quot;PN0287&quot; </code></pre>
<p>There were only 6 files for HER2+ breast cancer patients that had full RNA expression, so I did not perform this analysis.</p>
</div>
<div id="neoadjuvant-patient-characterization" class="section level1">
<h1>Neoadjuvant patient characterization</h1>
<pre class="r"><code>tempusneo &lt;- tempusmergedupclean %&gt;% filter(tempusmergedupclean$neoadj==1 &amp; !(StudyID==&quot;MB-0138&quot;) &amp; !(StudyID==&quot;MB-1020&quot;) &amp; !(StudyID==&quot;MB-1049&quot;))

kable(table(tempusneo$recurrenceposneg), caption = &quot;Neoadjuvant patients who experienced recurrence&quot;, col.names=c(&quot;Recurrence&quot;, &quot;Freq&quot;))</code></pre>
<table>
<caption>Neoadjuvant patients who experienced recurrence</caption>
<thead>
<tr class="header">
<th align="left">Recurrence</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">8</td>
</tr>
</tbody>
</table>
<pre class="r"><code>kable(table(tempusneo$vitalstatussurv), caption = &quot;Neoadjuvant patients who did not survive (0=died)&quot;, col.names=c(&quot;Vital status&quot;, &quot;Freq&quot;))</code></pre>
<table>
<caption>Neoadjuvant patients who did not survive (0=died)</caption>
<thead>
<tr class="header">
<th align="left">Vital status</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">8</td>
</tr>
</tbody>
</table>
<pre class="r"><code>fitsurvival &lt;- survfit(Surv(time=tempusneo$timefu, as.numeric(tempusneo$vitalstatussurv)) ~tempusneo$TP53YN, 
               data=tempusneo)
ggsurvplot(fitsurvival, data=tempusneo, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, title=&quot;Survival of neoadjuvant patients by TP53 status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Neoadjuvant%20patients-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Neoadjuvant patients-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
<tr>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
</tr>
</tbody>
</table>
<p></details></p>
<pre class="r"><code>fitrecurrence &lt;- survfit(Surv(time=tempusneo$pfstimefu, as.numeric(tempusneo$pfs)) ~tempusneo$TP53YN,
               data=tempusneo)
ggsurvplot(fitrecurrence, data=tempusneo, risk.table=&quot;absolute&quot;, pval=TRUE, xlab = &quot;Time in years&quot;, xlim=c(0, 10), break.time.by=1, ylab=&quot;Recurrence&quot;, title=&quot;Likelihood of recurrence in neoadjuvant patients by TP53 mutation status&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Neoadjuvant%20patients-2.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of Neoadjuvant patients-2.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
885580a
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-28
</td>
</tr>
<tr>
<td style="text-align:left;">
2ecfa66
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-05-24
</td>
</tr>
<tr>
<td style="text-align:left;">
fbe8a8e
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-12
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="heterogeneity-between-sentinel-lymph-nodes-and-primary-tumors" class="section level1">
<h1>Heterogeneity between sentinel lymph nodes and primary tumors</h1>
<pre class="r"><code>tempusbreastlymph &lt;- tempusmerge %&gt;% filter((StudyID %in% duplicates$StudyID) &amp; !(Tumorspecimen==&quot;&quot;) &amp; !(StudyID==&quot;PN0263&quot;))
  subtypelymph &lt;- tempusbreastlymph %&gt;% filter(Tumorspecimen==&quot;Lymph node&quot;)
  subtypebreast &lt;- tempusbreastlymph %&gt;% filter(Tumorspecimen==&quot;Breast&quot;)

    allgenesbreast&lt;- colSums(!((subtypebreast[,105:663])==&quot;&quot;))
  allgenesbreast[&quot;ERBB2/HER2&quot;] &lt;- sum(allgenesbreast[&quot;ERBB2&quot;], allgenesbreast[&quot;ERBB2 (HER2)&quot;], allgenesbreast[&quot;ERBB2(HER2)&quot;], allgenesbreast[&quot;ERBB2HER2)&quot;])
  print(head(sort(allgenesbreast, decreasing=TRUE)))</code></pre>
<pre><code> TP53  FAT1  MCL1 ZFHX3  ELF3 GATA1 
    5     4     4     4     3     3 </code></pre>
<pre class="r"><code>  allgeneslymph&lt;- colSums(!((subtypelymph[,105:663])==&quot;&quot;))
  allgeneslymph[&quot;ERBB2/HER2&quot;] &lt;- sum(allgeneslymph[&quot;ERBB2&quot;], allgeneslymph[&quot;ERBB2 (HER2)&quot;], allgeneslymph[&quot;ERBB2(HER2)&quot;], allgeneslymph[&quot;ERBB2HER2)&quot;])
  print(head(sort(allgeneslymph, decreasing=TRUE)))</code></pre>
<pre><code>PIK3CA   TP53 ARID1A ARID5B  ASXL1    ATM 
     4      2      1      1      1      1 </code></pre>
<p>We did see heterogeneity between the mutations represented in the lymph nodes and the mutations represented in the primary breast tumors for those samples. Two of the lymph nodes had 20% purity, 2 had 40% purity and 2 had 50% purity. This is a preliminary anaylsis.</p>
</div>
<div id="extremely-mutated-patients-x-2---case-examples" class="section level1">
<h1>Extremely mutated patients x 2 - case examples</h1>
<pre class="r"><code>tempusmergedupclean$StudyID[tempusmergedupclean$genecount==max(tempusmergedupclean$genecount)]</code></pre>
<pre><code>[1] PN0268
127 Levels: MB-0009 MB-0038 MB-0084 MB-0118 MB-0122 MB-0132 ... PN0308</code></pre>
<p>The patient who had the most unique genes mutated in her tumor in our sample (with 184) was black, was living as of last f/u with iniital diagnosis in 2001 and f/u in 2011. HR+/HER2- disease without known recurrence. Her TMB was 71.3 and in the 100th percentile, MSI status was equivocal. The patient who had the highest number of variants (9) within a single gene was white, was dead with initial diagnosis in 2000 and f/u in 2006 with HR+, HER2 unknown disease with unknown recurrence status. Her TMB was 58.3 and in the 100th percentile; genecount was 3rd highest in the group, MSI status was normal. Both patients were cigarette smokers although cigarette smoking was not significantly associated with any of our mutation variables.</p>
<pre class="r"><code>sameptsamegenemultivariant &lt;- tempusmutation %&gt;% group_by (StudyID, Gene) %&gt;% filter((duplicated(StudyID) &amp; duplicated(Gene) &amp; duplicated(AccessionNo)) | (duplicated(StudyID, fromLast=TRUE) &amp; duplicated(AccessionNo, fromLast=TRUE) &amp; duplicated(Gene, fromLast=TRUE))) %&gt;% dplyr::select(&quot;StudyID&quot;, &quot;AccessionNo&quot;, &quot;Tumorspecimen&quot;, &quot;Gene&quot;, &quot;variant_name_C&quot;, &quot;variant_name_P&quot;, &quot;NM_number&quot;, &quot;typeofvariant&quot;, &quot;genecount&quot;, &quot;variantcount&quot;)</code></pre>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.6

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] bindrcpp_0.2.2        ComplexHeatmap_1.20.0 stringr_1.3.1        
 [4] survminer_0.4.3       ggpubr_0.2            magrittr_1.5         
 [7] survival_2.43-3       ggplot2_3.1.0         knitr_1.21           
[10] dplyr_0.7.8           tidyr_0.8.2          

loaded via a namespace (and not attached):
 [1] shape_1.4.4         circlize_0.4.5      GetoptLong_0.1.7   
 [4] zoo_1.8-4           tidyselect_0.2.5    xfun_0.4           
 [7] purrr_0.2.5         splines_3.5.0       lattice_0.20-38    
[10] colorspace_1.4-0    generics_0.0.2      htmltools_0.3.6    
[13] yaml_2.2.0          survMisc_0.5.5      rlang_0.3.1        
[16] R.oo_1.22.0         pillar_1.3.1        glue_1.3.0         
[19] withr_2.1.2         R.utils_2.7.0       RColorBrewer_1.1-2 
[22] bindr_0.1.1         plyr_1.8.4          munsell_0.5.0      
[25] gtable_0.2.0        workflowr_1.1.1     R.methodsS3_1.7.1  
[28] GlobalOptions_0.1.0 evaluate_0.12       labeling_0.3       
[31] highr_0.7           broom_0.5.1         Rcpp_1.0.0         
[34] xtable_1.8-3        scales_1.0.0        backports_1.1.3    
[37] jsonlite_1.6        cmprsk_2.2-7        km.ci_0.5-2        
[40] gridExtra_2.3       rjson_0.2.20        digest_0.6.18      
[43] stringi_1.2.4       KMsurv_0.1-5        cowplot_0.9.4      
[46] rprojroot_1.3-2     tools_3.5.0         lazyeval_0.2.1     
[49] tibble_2.0.1        crayon_1.3.4        whisker_0.3-2      
[52] pkgconfig_2.0.2     Matrix_1.2-15       data.table_1.12.0  
[55] assertthat_0.2.0    rmarkdown_1.11      R6_2.3.0           
[58] nlme_3.1-137        git2r_0.24.0        compiler_3.5.0     </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
