<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Tempus Data Analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tempus 2019 Data Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Tempus.html">Tempus Data Analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a>
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Tempus Data Analysis</h1>

</div>


<p><strong>Last updated:</strong> 2019-02-10</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20190207)</code> </summary></p>
<p>The command <code>set.seed(20190207)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 445bd8d </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    code/
    Ignored:    data/
    Ignored:    docs/.DS_Store
    Ignored:    output/

Untracked files:
    Untracked:  docs/figure/

Unstaged changes:
    Modified:   analysis/Tempus.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
445bd8d
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-08
</td>
<td style="text-align:left;">
analysis 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
0333ca2
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
website build 1
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
9620b54
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
f7d6011
</td>
<td style="text-align:left;">
Sheila Rajagopal
</td>
<td style="text-align:left;">
2019-02-07
</td>
<td style="text-align:left;">
start site
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="data-structure-and-format" class="section level1">
<h1>Data structure and format</h1>
<pre class="r"><code>tempusmutationtemp$GeneInfo &lt;- paste (tempusmutationtemp$variantcount, tempusmutationtemp$variant_name_C, tempusmutationtemp$variant_name_P, tempusmutationtemp$NM_number, tempusmutationtemp$typeofvariant, tempusmutationtemp$VAF, tempusmutationtemp$COSMICCases, sep=&#39;\t&#39;) # combines variant-specific variables into a temporary column for now

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo) %&gt;%mutate(maxvariantcount = max(variantcount))

tempusmutationtemp &lt;- dplyr::select(tempusmutationtemp, -c(biologicalmeaning, variant_name_C, variant_name_P, NM_number, typeofvariant, VAF, COSMICCases, variantcount, Notes)) #drops excess columns for merge purposes

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% spread(Gene,GeneInfo) #Creates individual columns for each gene with the Gene Information as the value, every variant has its own row

tempusmutationtemp$V1=NULL #Clears artificial columns that were created in the process of formatting the dataset
tempusmutationtemp$X=NULL

tempusmutationtemp &lt;- tempusmutationtemp %&gt;% group_by(StudyID, AccessionNo, PanelName, Normalspecimen, Normalcollecteddate, Normalreceiveddate, Tumorspecimen, Tumorcollecteddate, Tumorreceiveddate, TumorPercentage, TumorMutationalBurden, PercentileTumorMutationalBurd, MicrosatelliteInstabilityStatu, DateSigned, genecount, maxvariantcount) %&gt;% summarise_all(funs(toString(na.omit(.), sep=&#39;\t&#39;))) #This flattens all of the rows -&gt; RIGHT NOW each patient is on one line and multiple mutations for the same gene are in one entry as are their variant information for ease of merge

tempusmerge &lt;- merge (tempusclinical, tempusmutationtemp, by=&quot;StudyID&quot;, all.x=TRUE, all.y=FALSE, sort=FALSE) #Merging the clinical and mutational data into one database, reports are excluded if they do not have clinical information</code></pre>
</div>
<div id="data-cleaningnew-variable-generation" class="section level1">
<h1>Data cleaning/new variable generation</h1>
<pre class="r"><code>tempusmerge$age50 &lt;- ifelse(tempusmerge$age_at_diagnosis &lt; 50, 1, 0) #binary variable for age &lt; 50 (which equals 1)
tempusmerge$neoadj &lt;- ifelse ((as.Date(as.character(tempusmerge$date_surg1), format=&quot;%m/%d/%Y&quot;)-as.Date(as.character(tempusmerge$date_cx1), format=&quot;%m/%d/%Y&quot;))&gt;0, 1, 0) #determining if pts had neoadjuvant chemotherapy
tempusmerge$HRstatus &lt;- ifelse (tempusmerge$ER==&quot;Pos&quot; | tempusmerge$PR==&quot;Pos&quot;, 1, 0) #Hormone-receptor positive status
tempusmerge$nodeposneg &lt;- ifelse (tempusmerge$RegionalNodePositive&gt;0, 1, 0) #At least one positive lymph node
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;00 Patient became disease-free after treatment and has not had a recurrence.&quot;, 0, 1) #Positive/negative recurrence status alone without differentiation of site. 
tempusmerge$recurrenceposneg &lt;- ifelse (tempusmerge$type1strecurrence==&quot;99 It is unknown whether the disease has recurred or if the patient was ever disease-free&quot;, NA, tempusmerge$recurrenceposneg) #Cleaning unknown recurrence variables from this

tempusmerge$TMB &lt;- as.numeric(tempusmerge$TumorMutationalBurden) #Numeric interpretation of mutational burden
tempusmerge$genemutshigh &lt;- ifelse (tempusmerge$genecount&gt;25, 1,0) #Patients with a high number of mutated genes as measured by number of unique genes mutated 

tempusmergedupclean &lt;- tempusmerge
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TumorPercentage, StudyID, FUN=max)==TumorPercentage),] #keep duplicates by higher %
tempusmergedupclean &lt;- tempusmergedupclean[with(tempusmergedupclean, ave(TMB, StudyID, FUN=max)==TMB),] #keep duplicates by higher TMB if same source, %</code></pre>
</div>
<div id="data-summary" class="section level1">
<h1>Data summary</h1>
<p>We have clinical information for 127 patients and mutation information for 131 patients.</p>
<p>We are missing clinical information for the following 4 patients. These are not included in the merged data set.</p>
<pre><code>[1] &quot;PN0118&quot; &quot;PN0124&quot; &quot;PN0153&quot; &quot;PN0155&quot;</code></pre>
<pre class="r"><code>duplicates &lt;- tempusreports %&gt;% filter(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE)) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
duplicates</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0139","2":"TL-18-1FFA96","3":"Breast","4":"0.2"},{"1":"PN0139","2":"TL-18-4BC15A","3":"Lymph node","4":"0.5"},{"1":"PN0198","2":"TL-18-200CF9","3":"Breast","4":"0.6"},{"1":"PN0198","2":"TL-18-C9E469","3":"Lymph node","4":"0.5"},{"1":"PN0264","2":"TL-18-25A755","3":"","4":"0.3"},{"1":"PN0264","2":"TL-18-18EB33","3":"","4":"0.2"},{"1":"PN0145","2":"TL-18-12B92C","3":"Breast","4":"0.4"},{"1":"PN0145","2":"TL-18-CDF9A7","3":"Lymph node","4":"0.4"},{"1":"PN0129","2":"TL-18-2F1FB7","3":"Breast","4":"0.9"},{"1":"PN0129","2":"TL-18-9BFB83","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CE9403","3":"Lymph node","4":"0.2"},{"1":"PN0221","2":"TL-18-CD4FA2","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-18-213530","3":"Breast","4":"0.6"},{"1":"PN0147","2":"TL-19-85F69C","3":"Lymph node","4":"0.4"},{"1":"PN0263","2":"TL-18-56509D","3":"Breast","4":"0.6"},{"1":"PN0263","2":"TL-18-8A2A03","3":"Breast","4":"0.6"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>8 patients have duplicate reports (most often primary tumor and lymph node) submitted with the above characteristics. These patients are included in the merged data set but in these cases, for any analysis, we counted each patient individually/once using the specimen that offered the best tumor percentage as relevant.</p>
<pre class="r"><code>lymphnodes &lt;- tempusreports %&gt;% filter((Tumorspecimen==&quot;Lymph node&quot;) &amp; !(duplicated(StudyID) | duplicated(StudyID, fromLast = TRUE))) %&gt;% select(StudyID, AccessionNo, Tumorspecimen, TumorPercentage)
lymphnodes</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["StudyID"],"name":[1],"type":["fctr"],"align":["left"]},{"label":["AccessionNo"],"name":[2],"type":["fctr"],"align":["left"]},{"label":["Tumorspecimen"],"name":[3],"type":["fctr"],"align":["left"]},{"label":["TumorPercentage"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"PN0162","2":"TL-18-BDD8C3","3":"Lymph node","4":"0.20"},{"1":"PN0217","2":"TL-18-DD82EF","3":"Lymph node","4":"0.20"},{"1":"PN0255","2":"TL-18-5E23D5","3":"Lymph node","4":"0.85"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>3 patients have only lymph node submitted. These patients are included in the merged data set and treated as though they had primary tumor (as this was their only tumor specimen).</p>
</div>
<div id="clinical-summary" class="section level1">
<h1>Clinical summary</h1>
<p>Low-yield variables: Date and age of enrollment (38 patients), Sex (1 male patient), Insurance (1 patient with unknown insurance, 126 patients insured), Alcohol history (90 patients not reported), gravidity/parity (26 patients reported), menopause (26 patients reported), distant metastatic sites (only 2 sites reported), histology (14 patients with at least partial lobular disease), surgical margin (4 patients with ? residual disease), 2nd primary (only one patient reported)</p>
<p>Variables not incorporated for this analysis: Marital status at diagnosis, breast primary site, laterality, TNM edition and clinical staging, tumorsize, lymphovascularinvasion (64 patients only reported), radiation variables</p>
<p><img src="figure/Tempus.Rmd/Clinical%20summary-1.png" width="672" style="display: block; margin: auto;" /> This is the age distribution of patients at time of diagnosis. This population skews young but does have representation throughout the age range.</p>
<table>
<caption>Distribution of patients younger than age 50 (with 0 &gt; 50 and 1 &lt; 50</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">70</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">57</td>
</tr>
</tbody>
</table>
<table>
<caption>Self-reported races in our cohort (Y/N refers to Hispanic ethnicity)</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">No</th>
<th align="right">Yes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Asian</td>
<td align="right">1</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td>Black</td>
<td align="right">60</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td>White</td>
<td align="right">63</td>
<td align="right">3</td>
</tr>
</tbody>
</table>
<table>
<caption>Tobacco exposure</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">00 Never used</td>
<td align="right">62</td>
</tr>
<tr class="odd">
<td align="left">01 Cigarette smoker</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">05 Previous tobacco use, but not at present time</td>
<td align="right">26</td>
</tr>
<tr class="odd">
<td align="left">99 Unknown</td>
<td align="right">20</td>
</tr>
</tbody>
</table>
<table>
<caption>Alive or dead status last known</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0 Dead</td>
<td align="right">46</td>
</tr>
<tr class="even">
<td align="left">1 Alive</td>
<td align="right">81</td>
</tr>
</tbody>
</table>
<table>
<caption>Fraction of patients with at least one node positive on study</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">51</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">76</td>
</tr>
</tbody>
</table>
<table>
<caption>Ki67 distribution as known in 116/127 patients</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">116</td>
</tr>
<tr class="even">
<td align="left">&gt;90%</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">10%</td>
<td align="right">3</td>
</tr>
<tr class="even">
<td align="left">20-30%</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">20%</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">30%</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">35%</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">50%</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">60%</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">70%</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<table>
<caption>Distribution of analytical stage based on a synthesis of clinical stage and pathological stage information</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="right">16</td>
</tr>
<tr class="even">
<td align="left">1A</td>
<td align="right">15</td>
</tr>
<tr class="odd">
<td align="left">1B</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">2A</td>
<td align="right">37</td>
</tr>
<tr class="odd">
<td align="left">2B</td>
<td align="right">28</td>
</tr>
<tr class="even">
<td align="left">3</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">3A</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">3B</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">3C</td>
<td align="right">5</td>
</tr>
<tr class="even">
<td align="left">4</td>
<td align="right">2</td>
</tr>
</tbody>
</table>
<table>
<caption>Hormone receptor status (y-axis) vs HER2 status (x-axis)</caption>
<thead>
<tr class="header">
<th></th>
<th></th>
<th align="right">Neg</th>
<th align="right">Pos</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0</td>
<td>1</td>
<td align="right">31</td>
<td align="right">12</td>
</tr>
<tr class="even">
<td>1</td>
<td>4</td>
<td align="right">53</td>
<td align="right">26</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of chemotherapy</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">38</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">89</td>
</tr>
</tbody>
</table>
<table>
<caption>Patients with neoadjuvant therapy</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0</td>
<td align="right">61</td>
</tr>
<tr class="even">
<td align="left">1</td>
<td align="right">27</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of radiotherapy</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">49</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">78</td>
</tr>
</tbody>
</table>
<table>
<caption>Initial surgery performed at time of diagnosis and initial treatment</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">BCS</td>
<td align="right">48</td>
</tr>
<tr class="odd">
<td align="left">bilateral mam</td>
<td align="right">8</td>
</tr>
<tr class="even">
<td align="left">mastectomy</td>
<td align="right">68</td>
</tr>
<tr class="odd">
<td align="left">other/uk/nos</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<table>
<caption>Recieved any type of hormone therapy</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No</td>
<td align="right">56</td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="right">71</td>
</tr>
</tbody>
</table>
<p>Of note, with mutation testing prior to sequencing, we had 1 patient each report a BRCA 1 and BRCA 2 mutation.</p>
</div>
<div id="mutations-summary" class="section level1">
<h1>Mutations summary</h1>
<p>Variables not incorporated for this analysis: Percentile mutational burden (incomplete)</p>
<p>We have representation of 560 genes with mutations in this data set (including both primary tumors and lymph nodes). The most common mutations were as follows:</p>
<pre><code>  TP53  KMT2D  ZFHX3  KMT2B PIK3CA   MCL1 ARID1A NOTCH1 BCORL1   SPEN 
    90     74     68     55     53     52     37     37     35     34 </code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20summary%20and%20tumor%20purity-1.png" width="672" style="display: block; margin: auto;" /> This is the tumor purity distribution of patients at time of diagnosis. 30% purity is conisdered sufficient to perform MSI testing, for example. 104 of our 127 samples met this criteria.</p>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   1.00    4.00   17.00   16.08   24.50   41.00 </code></pre>
<p><img src="figure/Tempus.Rmd/Ongoing%20mutational%20summary-1.png" width="672" style="display: block; margin: auto;" /> Only 88 of 139 reports contained percentile information.</p>
<table>
<caption>Distribution of MSI status</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Equivocal</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">High</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">MSI can not be assessed because the tumor content is below 30%.</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">Stable</td>
<td align="right">64</td>
</tr>
<tr class="odd">
<td align="left">Stable (MSS)</td>
<td align="right">43</td>
</tr>
<tr class="even">
<td align="left">Undetermined</td>
<td align="right">6</td>
</tr>
</tbody>
</table>
<table>
<caption>Distribution of variant types across all mutations</caption>
<thead>
<tr class="header">
<th align="left">Var1</th>
<th align="right">Freq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Missense variant</td>
<td align="right">1807</td>
</tr>
<tr class="even">
<td align="left">Stop gain</td>
<td align="right">412</td>
</tr>
<tr class="odd">
<td align="left">Splice region variant</td>
<td align="right">407</td>
</tr>
<tr class="even">
<td align="left">Copy number gain</td>
<td align="right">319</td>
</tr>
<tr class="odd">
<td align="left">Copy number loss</td>
<td align="right">319</td>
</tr>
<tr class="even">
<td align="left">Frameshift</td>
<td align="right">255</td>
</tr>
<tr class="odd">
<td align="left">Frameshift- LOF</td>
<td align="right">64</td>
</tr>
<tr class="even">
<td align="left">Stop gain- LOF</td>
<td align="right">57</td>
</tr>
<tr class="odd">
<td align="left">Inframe deletion</td>
<td align="right">51</td>
</tr>
<tr class="even">
<td align="left">Splice region</td>
<td align="right">36</td>
</tr>
<tr class="odd">
<td align="left">Missense variant- LOF</td>
<td align="right">29</td>
</tr>
<tr class="even">
<td align="left">Missense variant - LOF</td>
<td align="right">26</td>
</tr>
<tr class="odd">
<td align="left">Frameshift - LOF</td>
<td align="right">22</td>
</tr>
<tr class="even">
<td align="left">copy number loss</td>
<td align="right">19</td>
</tr>
<tr class="odd">
<td align="left">Stop gain - LOF</td>
<td align="right">19</td>
</tr>
<tr class="even">
<td align="left">Splice region variant- LOF</td>
<td align="right">18</td>
</tr>
<tr class="odd">
<td align="left">copy number gain</td>
<td align="right">17</td>
</tr>
<tr class="even">
<td align="left">Missense variant (exon 10) - GOF</td>
<td align="right">17</td>
</tr>
<tr class="odd">
<td align="left">missense variant</td>
<td align="right">14</td>
</tr>
<tr class="even">
<td align="left">Missense variant (exon 21) - GOF</td>
<td align="right">13</td>
</tr>
<tr class="odd">
<td align="left">Copy Number gain</td>
<td align="right">11</td>
</tr>
<tr class="even">
<td align="left">Missense variant- GOF</td>
<td align="right">10</td>
</tr>
<tr class="odd">
<td align="left">region variant</td>
<td align="right">8</td>
</tr>
<tr class="even">
<td align="left">Start loss</td>
<td align="right">8</td>
</tr>
<tr class="odd">
<td align="left">frameshift</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">Frameshift VAF:</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">Inframe insertion</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">Missense variant -LOF</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">Missense variant (exon 5) - GOF</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">Splice region vairant</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">Splice region variant- GOF</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">Splice region variant - LOF</td>
<td align="right">3</td>
</tr>
<tr class="even">
<td align="left">stop gain</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">Stop gain-</td>
<td align="right">3</td>
</tr>
<tr class="even">
<td align="left">Variant</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">Copy Number Loss</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Frameshift LOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Missense Variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Missense variant - GOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Missense variant (exon 10)- GOF</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Missense variant (exon 14) - GOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Missense variant â LOF</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Missense variant LOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Missense variant-LOF</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Missense variant(exon 2) - GOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">splice region variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Splice region variant â LOF</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Splice region variant VAF</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Stop Gain</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">Stop gain LOF</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">Copy numberloss</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">copy numbre gain</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Frameshift- LOF VAF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">frameshift-LOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">FrameshiftVAF:</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Inframe deletion - GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">inframe insertion</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Inframe insertion - GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Missense variant</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missense varian(exon 10) - GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Missense variant -</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missense variant (exon 21) - GOF VAF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Missense variant (exon 8) - GOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missense Variant (exon 8)- GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Missense variant GOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missense variant(exon 10) - GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Missense variant(exon 4) - GOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missesne variant</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">rameshift</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">region variant- LOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Splice</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Splice region vairant - LOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Splice region variant -LOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Splice region variant â GOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Splice region variant-(exon 3) â GOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Start Loss</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">StartStop gain loss- LOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Stop gain VAF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Stop gain- GOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Stop gain-LOF</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Stop gain(exon 4) - LOF</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Stop loss</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Stopgain</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">Missense variants, splice region variants</td>
<td align="right">, stop gains and copy number gains/losses were the best represented in our cohort.</td>
</tr>
</tbody>
</table>
</div>
<div id="exploring-what-mutation-distribution-means-from-a-clinical-standpoint" class="section level1">
<h1>Exploring what mutation distribution means from a clinical standpoint</h1>
<pre class="r"><code>summary(tempusmergedupclean$genecount)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   1.00    7.00   12.00   24.72   24.00  184.00 </code></pre>
<pre class="r"><code>table(tempusmergedupclean$genecount&gt;25)</code></pre>
<pre><code>
FALSE  TRUE 
   96    31 </code></pre>
<pre class="r"><code>hist(tempusmergedupclean$genecount, main=&quot;Distribution of uniquely mutated genes per patient sample&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20distribution%20per%20patient-1-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>hist(tempusmergedupclean$maxvariantcount, main=&quot;Distribution of highest number of variants\n for a single gene per patient sample&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20distribution%20per%20patient-1-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>hist(tempusmergedupclean$TMB, main=&quot;Distribution of tumor mutational burden (as measured by Tempus)&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20distribution%20per%20patient-1-3.png" width="672" style="display: block; margin: auto;" /> We see a long tail towards the right in terms of frequency of uniquely mutated genes, number of variants for a single gene and TMB. However, these do not clearly correlate in a linear distribution.</p>
<pre class="r"><code>ggplot()+
  geom_line(data=tempusmergedupclean, aes(x=tempusmergedupclean$TMB, y=tempusmergedupclean$maxvariantcount), size=1, color=&quot;red&quot;)+
  geom_line(data=tempusmergedupclean, aes(x=tempusmergedupclean$TMB, y=tempusmergedupclean$genecount), size=1, color=&quot;blue&quot;)</code></pre>
<p><img src="figure/Tempus.Rmd/Mutation%20distribution%20per%20patient-2-1.png" width="672" style="display: block; margin: auto;" /> There are peaks and valleys with rising TMB in which gene count and max variant count (per gene) correlate to each other, but these do not increase in a completely linear fashion.</p>
<pre class="r"><code>chisq.test((tempusmergedupclean$genecount&gt;12), tempusmergedupclean$recurrenceposneg)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  (tempusmergedupclean$genecount &gt; 12) and tempusmergedupclean$recurrenceposneg
X-squared = 2.374, df = 1, p-value = 0.1234</code></pre>
<pre class="r"><code>chisq.test((tempusmergedupclean$maxvariantcount&gt;2), tempusmergedupclean$recurrenceposneg)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  (tempusmergedupclean$maxvariantcount &gt; 2) and tempusmergedupclean$recurrenceposneg
X-squared = 1.686, df = 1, p-value = 0.1941</code></pre>
<pre class="r"><code>chisq.test((tempusmergedupclean$TMB&gt;3), tempusmergedupclean$recurrenceposneg)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  (tempusmergedupclean$TMB &gt; 3) and tempusmergedupclean$recurrenceposneg
X-squared = 5.8608, df = 1, p-value = 0.01548</code></pre>
<p>Using the Chi-squared test, there was an association between tumor mutational burden as measured by Tempus and recurrence at all, but not using number of genes mutated (above the mean) or maximum number of variants for a single gene (above the mean).</p>
<pre class="r"><code>#Do copy number changes and splice variants affect outcome</code></pre>
<pre class="r"><code>#HER2 and HRstatus discrepanies
#HER2 + but 19 copy number change in total (discrepancy between how patients were treated and longterm) 

#kable(table(tempusmerge$HRstatus[unique(tempusmerge$StudyID)], tempusmerge$hormonetherapy[unique(tempusmerge$StudyID)]), caption=&quot;Fraction of patients who received hormone therapy relative to having hormone receptor status positive disease - 15 patients did not get hormone therapy when they did have hormone receptor positive disease reportedly&quot;)</code></pre>
<pre class="r"><code>#Neoadjuvant characteristic as attribute -&gt; look at patterns of response based on mutations (ER+ / ER-) take out HER2+ </code></pre>
<pre class="r"><code>#For triple negative patients, see if there is residual disease after chemotherapy?
#Look at what mutations predict response if neoadjuvant?</code></pre>
<pre class="r"><code>#Dates of surgery OK to start / enter into the study
#Alive or dead / date of last f/u</code></pre>
<pre class="r"><code>#sameptsamegenemultivariant &lt;- tempusmutation %&gt;% group_by (StudyID, Gene) %&gt;% filter((duplicated(StudyID) &amp; duplicated(Gene) &amp; duplicated(AccessionNo)) | (duplicated(StudyID, fromLast=TRUE) &amp; duplicated(AccessionNo, fromLast=TRUE) &amp; duplicated(Gene, fromLast=TRUE))) %&gt;% dplyr::select(&quot;StudyID&quot;, &quot;AccessionNo&quot;, &quot;Tumorspecimen&quot;, &quot;Gene&quot;, &quot;variant_name_C&quot;, &quot;variant_name_P&quot;, &quot;NM_number&quot;, &quot;typeofvariant&quot;, &quot;genecount&quot;, &quot;variantcount&quot;)</code></pre>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] bindrcpp_0.2.2 ggplot2_3.1.0  knitr_1.21     dplyr_0.7.8   
[5] tidyr_0.8.2   

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0        highr_0.7         pillar_1.3.1     
 [4] compiler_3.5.0    git2r_0.24.0      plyr_1.8.4       
 [7] workflowr_1.1.1   bindr_0.1.1       R.methodsS3_1.7.1
[10] R.utils_2.7.0     tools_3.5.0       digest_0.6.18    
[13] jsonlite_1.6      evaluate_0.12     tibble_2.0.1     
[16] gtable_0.2.0      pkgconfig_2.0.2   rlang_0.3.1      
[19] yaml_2.2.0        xfun_0.4          withr_2.1.2      
[22] stringr_1.3.1     rprojroot_1.3-2   grid_3.5.0       
[25] tidyselect_0.2.5  glue_1.3.0        R6_2.3.0         
[28] rmarkdown_1.11    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.3   scales_1.0.0     
[34] htmltools_0.3.6   assertthat_0.2.0  colorspace_1.4-0 
[37] labeling_0.3      stringi_1.2.4     lazyeval_0.2.1   
[40] munsell_0.5.0     crayon_1.3.4      R.oo_1.22.0      </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
